<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 
 <html>
 <head>
 
 <title>58. Alignment of Promoter Regions by Mapping Nucleotide Sequences into Arrays of Transcription Factors Binding Motifs</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category02.html'>&lt;&lt;</a> <a href='/recombdb/category03.html'><b>Recognition of Genes and Regulatory Elements</b> Overview</a> <a href='/recombdb/category04.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-02-057/'>&lt;&lt;</a> Poster 058 [ <a href='poster-03-058.pdf'>PDF</a> | <a href='poster-03-058.tgz'>TeX</a> | <a href='poster-03-058.ps'>PostScript</a> ] <a href='/recombdb/poster-03-059/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>58. Alignment of Promoter
 Regions by Mapping Nucleotide Sequences into Arrays of
 Transcription Factors Binding Motifs<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font>
 <table cellpadding="3">
 <tr>
 <td align="center"><b>E. Blanco<img width="21" height="18"
      align="bottom" border="0" src="img1.png" alt=
      "$^{1,2}$"></b></td>
 <td align="center"><b>X. Messeguer<img width="11" height="18"
      align="bottom" border="0" src="img2.png" alt="$^2$"></b></td>
 <td align="center"><b>R. Guig&oacute;<img width="21" height="18"
 align="bottom" border="0" src="img3.png" alt=
 "$^{1,\star}$"></b></td></tr>
 <tr>
 <td align="center"><b><tt>eblanco@imim.es</tt></b></td>
 <td align="center"><b><tt>peypoch@lsi.upc.es</tt></b></td>
 <td align="center">
 <b><tt>rguigo@imim.es</tt></b></td></tr></table></div>
 <hr>
  <font size="-1"><br>
 <b>1:</b> Grup de Recerca en Inform&agrave;tica Biom&egrave;dica.
 Institut Municipal d'Investigaci&oacute; Medica / Universitat
 Pompeu Fabra / Centre de Regulaci&oacute; Gen&ograve;mica. Pg.
 Maritim de la Barceloneta 37-49, 08003 Barcelona, Spain. <b>2:</b>
 Grup d'algor&iacute;smica i gen&egrave;tica. Departament de
 Llenguatges i Sistemes Inform&agrave;tics. Universitat
 Polit&egrave;cnica de Catalunya. C/ Jordi Girona, 1-3, 08034
 Barcelona, Spain. <b><img width="12" height="13" align="bottom"
 border="0" src="img4.png" alt="$\star$">:</b> Corresponding
 author.<br></font> 
 <p><font size="-1"><font size="-1"><b>Keywords:</b> Sequence
 analysis, dynamic programming, comparative genomics, gene
 expression.</font></font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Introduction</font></a></h1>
 <font size="-1"><font size="-1">We address the problem of comparing
 promoter regions from genes with similar expression patterns (e.g.
 homologous genes, [<a href="#DC02">1</a>]).
 Similarity in gene expression may not be reflected in sequence
 similarity in promoter regions which partially would explain the
 limited sucess of currently available computational methods for
 promoter prediction ([<a href="#FW00">2</a>]). In
 the approach described here, we attempt to overcome such a
 limitation representing a (potential) promoter region as a sequence
 in a new alphabet in which the different symbols denote different
 Transcription Factors. Thus, a promoter can be translated into a
 sequence of pairs containing the factor potentially binding to a
 motif and the associated position on the nucleotide sequence.
 Sequences in this new alphabet can be aligned. If the scoring model
 takes into account not only the presence/absence of a given symbol,
 but its relative position on the primary sequence as well, the
 optimal alignment between the promoter regions of two similarly
 expressed genes may reflect the underlying common configuration of
 binding motifs.</font></font> 
 <h1><a name="SECTION00020000000000000000"><font size=
 "+1">Algorithm</font></a></h1>
  
 <p><font size="-1"><font size="-1">To obtain the optimal alignment
 between two sequences <img width="16" height="14" align="bottom"
 border="0" src="img5.png" alt="$A$"> and <img width="17"
      height="14" align="bottom" border="0" src="img6.png" alt=
      "$B$"> in the new alphabet, we introduce a dynamic programming
      algorithm similar to a method initially developed to compare
      enzyme restriction maps, [<a href=
      "#WSK84">3</a>]. Elements 
 <img width="101" height="29" align="middle" border="0" src=
 "img7.png" alt="$a_i = &lt;r_i,p_i&gt;$"> in <img width="16"
      height="14" align="bottom" border="0" src="img5.png" alt=
      "$A$"> and 
 <img width="101" height="30" align="middle" border="0" src=
 "img8.png" alt="$b_j = &lt;t_j,q_j&gt;$"> in <img width="17"
      height="14" align="bottom" border="0" src="img6.png" alt=
      "$B$"> are said to match if and only if <img width="50"
      height="29" align="middle" border="0" src="img9.png" alt=
      "$r_i = t_j$"> (the same TF). The optimal global alignment
      ending at match 
 <img width="56" height="30" align="middle" border="0" src=
 "img10.png" alt="$a_i \leftrightarrow b_j$"> is computed, then,
 according to the following recurrence:</font></font></p>
 <table width="2">
 <tr>
 <td>][c] <b>if <img width="17" height="29" align="middle"
      border="0" src="img11.png" alt="$r_i$"> = <img width="16"
      height="29" align="middle" border="0" src="img12.png" alt=
      "$t_j$"> then<br>
 <br>
  
 <img width="577" height="32" align="middle" border="0" src=
 "img13.png" alt=
 "$~~~~~~~~D(a_i,b_j) \equiv D_{ij} = min_{k,l} \{ D_{i-k,j-l} + \lambda (k+l-2) + \mu (p_i - p_{i-k} - q_j + q_{j-l})\}$">
 <br>
  <font size="-2">
 <img width="260" height="30" align="middle" border="0" src=
 "img14.png" alt=
 "$~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~0 &lt; k \leq i$"><br>
  
 <img width="258" height="30" align="middle" border="0" src=
 "img15.png" alt=
 "$~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~0 &lt; l \leq j$"></font><br></b>
 otherwise<br>
  
 <img width="108" height="30" align="middle" border="0" src=
 "img16.png" alt="$~~~~~~~~D_{ij} = \infty$"><br>
  <b>Initialization:<br>
  
 <img width="473" height="30" align="middle" border="0" src=
 "img17.png" alt=
 "$~~~~~~~~D_{00} = 0 ~~~~~~~~~~ D_{i0} = D_{i,m+1} = \infty ~~~~~~~~~~ D_{0j} = D_{n+1,j} = \infty$">
 </b></td></tr></table>
  
 <p><font size="-1"><font size="-1">where <img width="13"
      height="14" align="bottom" border="0" src="img18.png" alt=
      "$\lambda$"> and <img width="14" height="29" align="middle"
      border="0" src="img19.png" alt="$\mu$"> are two parameters to
      control the length and the compactness of the final alignment.
      If <img width="16" height="14" align="bottom" border="0" src=
      "img5.png" alt="$A$"> and <img width="17" height="14" align=
      "bottom" border="0" src="img6.png" alt="$B$"> contain
      <img width="14" height="13" align="bottom" border="0" src=
      "img20.png" alt="$n$"> elements, the cost of the algorithm is
      <img width="41" height="34" align="middle" border="0" src=
      "img21.png" alt="$\theta(n^4)$">. Because of the matrix is
      quite sparse, the final cost can be rewritten in terms of the
      number of matching positions between both lists <img width=
      "60" height="30" align="middle" border="0" src="img22.png"
      alt="$F &lt;&lt; n$"> as <img width="44" height="34" align=
      "middle" border="0" src="img23.png" alt=
      "$\theta(F^2)$">.</font></font></p>
 <h1><a name="SECTION00030000000000000000">Figures</a></h1>
  
 <div align="center"><a name="92"></a> 
 <table>
 <caption align="bottom"><strong>Figure:</strong> <font size=
 "-1">Alignment of the promoter regions from the homologues
 <small>HUMHBBAG</small> and <small>BOVHBB</small> of the
 <i>A-gamma-hemoglobin</i> gene.</font></caption>
 <tr>
 <td>
 <div align="center">
 <img width="503" height="228" align="bottom" border="0" src=
 "img24.png" alt=
 "\scalebox{0.96}[0.7]{ \includegraphics[bb=131 321 463 527]{poster-03-058-fig02.ps}}">
 </div></td></tr></table></div>
  
 <div align="center"><a name="94"></a> 
 <table>
 <caption align="bottom"><strong>Figure:</strong> <font size=
 "-1">Graphical representation of the TF sites predicted by the
 program MatInspector ([<a href="#QFK95">4</a>])
 and the results of the alignment along the human and mouse
 promoters of the gene <i>PS-PLA1</i>.</font></caption>
 <tr>
 <td>
 <div align="center">
 <img width="520" height="13" align="bottom" border="0" src=
 "img25.png" alt=
 "\scalebox{0.4}[0.265]{ \includegraphics[bb=25 25 566 820,angle=-90]{poster-03-058-fig01.ps}}">
 </div></td></tr></table></div>
  
 <p></p>
 <h2><a name="SECTION00040000000000000000">Bibliography</a></h2>
 <dl compact>
 <dt><a name="DC02">1</a></dt>
 <dd>Dermitzakis, E.T. and Clark, A.G. 2002. Evolution of
 transcription factor binding sites in mammalian gene regulatory
 regions: conservation and turnover. <em>Molecular Biology and
 Evolution</em> 19:1114-1121.</dd>
 <dt><a name="FW00">2</a></dt>
 <dd>Fickett, J.W. and Wasserman, W.W. 2000. Discovery and modelling
 of transcription regulatory regions. <em>Current opinion in
 Biotechnology</em> 11:19-24.</dd>
 <dt><a name="WSK84">3</a></dt>
 <dd>Waterman, M.S., Smith T.F. and Katcher, H.L. 1984. Algorithms
 for restriction map comparisons. <em>Nucleic Acids Research</em>
 12:237-242.</dd>
 <dt><a name="QFK95">4</a></dt>
 <dd>Quandt, K., Frech, K., Karas, H., Wingender, E. and Werner, T.
 1995. MatInd and MatInspector: new fast and versatile tools for
 detection of consensus matches in nucleotide sequence data.
 <em>Nucleic Acids Research</em> 23:4878-4884.</dd></dl>
  
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
