<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>61. Detection of Regulatory Elements by Comparative DNA Analysis</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category02.html'>&lt;&lt;</a> <a href='/recombdb/category03.html'><b>Recognition of Genes and Regulatory Elements</b> Overview</a> <a href='/recombdb/category04.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-03-060/'>&lt;&lt;</a> Poster 061 [ <a href='poster-03-061.pdf'>PDF</a> | <a href='poster-03-061.doc'>MSWORD</a> ] <a href='/recombdb/poster-03-062/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>61.&nbsp; Detection of Regulatory Elements<br>
  by Comparative DNA Analysis</h2>
 <p align="center"><b>Ulrike Gausmann<a href="#_ftn1" name=
 "_ftnref1" title="">[1]</a>,&nbsp; Niels Jahn<sup>1</sup>,&nbsp;
 Cornelia Platzer<a href="#_ftn2" name="_ftnref2" title=
 "">[2]</a>,<br>
  Karol Szafranski<sup>1</sup>,&nbsp; Matthias
 Platzer<sup>1</sup></b></p>
 <p><sup>&nbsp;</sup></p>
 <p><b>Keywords:</b> promoter, transcription factor binding sites,
 comparative sequence analysis</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction</b></p>
 <p>Within the SFB&nbsp;604 &lsquo;Multi&shy;functional Signaling
 Proteins&rsquo; (http://www.imb-jena.de/SFB604/) we deal with
 proteins which physically interact and may be potentially
 co-regulated.&nbsp; Comparative analysis of the 5'&nbsp;upstream
 gene regions should reveal regulatory elements common for all
 members of a group because functional elements are more strictly
 conserved and can be identified by align&shy;ment procedures.&nbsp;
 Sensitive pairwise align&shy;ment algorithms are the first steps
 but will not give clear results in many cases.&nbsp; For some
 purposes specia&shy;lized programs can be used, e.g. &lsquo;DNA
 block aligner&rsquo; (<i>DBA</i>) for the detection of conserved
 blocks in human and rodent orthologous non-coding sequences
 [1].</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</b>
 <b>Comparative promoter analysis of orthologs, paralogs, and
 co&#8209;regulated genes</b></p>
 <p>The most common method for the prediction of putative regulatory
 regions in candidate sequences is based on the comparison of the
 nucleotide sequence with known binding matrices for transcription
 factor binding sites (TFBS), e.g. with <i>MatInspector</i> or
 <i>MATCH</i>.&nbsp; However, these predictions still appear to be
 insufficient in sensitivity and specificity.&nbsp; Several
 improvements are on the way, namely data extension for matrix
 computation (<i>TransFac</i>), building of higher order binding
 site models (e.g. <i>TransCompel</i>), and the consideration of
 comparative approaches (e.g. <i>rVISTA</i> or <i>TOUCAN</i>).</p>
 <p>Pairwise comparisons were done with several local align&shy;ment
 programs and best results were obtained for orthologs from man and
 rodents with <i>DBA</i> (see below).&nbsp; Analyses of more than
 two corresponding regions was performed with the <i>HGFR</i> gene
 (hepatocyte growth factor receptor or c&#8209;met) where twelve
 genomic sequences (main source&nbsp; http:// www.nisc.nih.gov/
 projects/ zooseq.html) can be assembled so far.&nbsp; This gene
 contains no TATA box, but is associated with a CpG-island and seems
 to have a non-coding first exon separated by more than 20 kb from
 the second one.&nbsp; Reasonable alignments (<i>BlastZ</i>) were
 obtained with ten mammalian sequences after repeat-masking.&nbsp;
 50 kb of non-coding sequence upstream from the startcodon was used
 in each case.&nbsp; Regions with high similarity were extracted and
 analyzed with TFBS predicting programs.&nbsp; The results are
 compared with the output of <i>DBA</i> analyses of rodent and human
 sequences.</p>
 <p>As a representative of a set of paralogs we are investigating
 potassium channels where the subunit compositions are tissue
 dependent.&nbsp; Details will be shown at the conference.</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>The IL&#8209;10 gene as an
 example for comparative analysis of orthologs</b></p>
 <p>As an example for the applicability of the comparative approach
 we present the analysis of the Interleukin&#8209;10 (IL&#8209;10)
 promoter.&nbsp; IL&#8209;10 is a cytokine with
 immuno&shy;suppressive and anti-inflammatory properties and its
 synthesis can be induced or enhanced by various stimuli,
 particularly <i>via</i> a cAMP-dependent signaling pathway.&nbsp;
 Two functionally active CRE motifs, CRE1 and CRE4, were identified
 in the promoter region previously.&nbsp; It was suspected that
 other, non-CRE sites play also a role in this path&shy;way.&nbsp;
 Comparative analysis of three orthologous promoter sequences
 (<i>H.&nbsp;sapiens</i>, <i>M.&nbsp;musculus</i>,
 <i>M.&nbsp;monax</i>) with <i>DBA</i> and <i>MatInspector</i>
 revealed that two binding sites for the CCAAT enhancer binding
 protein, C/EBP3 and&nbsp;5, and the previously identified CRE4
 site, are located in conserved regions.&nbsp; Using EMSA and
 reporter gene assays the prominent functional role of these sites
 was experimentally confirmed.&nbsp; In contrast, the weakly
 conserved sites CRE1 and C/EBP1 were shown to be less important
 [2].</p>
 <p><b>4&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Further research
 directions</b></p>
 <p>The IL-10 example demonstrates the reasonability of close
 collaboration between bio&shy;informa&shy;ticians and experimental
 researchers.&nbsp; As the number of known genomes increases the
 back&shy;ground modeling of genomic sequences from different
 species will be more reliable and therefore, multiple
 align&shy;ment procedures will result in more significant
 predictions for non-coding DNA regions.&nbsp; Some approaches aim
 for analyses that can be performed with lengthy sequences where
 programs like <i>DBA</i> need too much computing time [3]. In the
 &lsquo;Jenaer Centre of Bioinformatics&rsquo; (http://
 www.imb-jena.de/ jcb/) we are evaluating parameters for
 align&shy;ment and prediction programs and are developing an
 analysis pipeline for sequence datasets.</p>
 <p>The evaluation and optimization of algorithms requires test- and
 training sets and the automated data transfer from local storage to
 analysis programs.&nbsp; Hence, we started to build a relational
 database which will also provide the experimental researchers
 within the SFB with data they need for their experiments, i.e.
 sequences with corresponding annotations derived from public
 references and from our analyses.&nbsp; To enhance the data basis
 we are including paralogous and orthologous gene sequences.</p>
 <h1>References</h1>
 <p>[1] Jareborg N., Birney E., and Durbin R. 1999. Comparative
 analysis of noncoding regions of 77 orthologous mouse and human
 gene pairs. <i>Genome Research</i> 9:815-824.</p>
 <p>[2] Brenner S., Pr&ouml;sch S., Schenke-Layland K., Riese U.,
 Gausmann U. and Platzer C. 2003. cAMP-induced IL&#8209;10 promoter
 activation depends on CCAAT/enhancer binding protein expression and
 monocytic differentiation. <i>Journal of Biological Chemistry, in
 press</i>.</p>
 <p>[3] Dieterich C., Cusack B., Wang H., Rateitschak K., Krause A.
 and Vingron M. 2002. Annotating regulatory DNA based on man-mouse
 genomic comparison. <i>Bioinformatics</i> 18 (ECCB 2002)
 S84-S90.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a> Department of
 Genome Analysis, Institute of Molecular Biotechnology, Postfach
 100813, 07708 Jena, Germany. E&#8209;mail:
 (ugau,nielsj,szafrans,mplatzer)@imb-jena.de</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Institute of
 Anatomy&nbsp;II, Medical School, Friedrich Schiller University
 Jena, Teichgraben&nbsp;7, 07740 Jena, Germany. E&#8209;mail:
 Cornelia.Platzer@mti.uni-jena.de</p></div></div>
 </body>
 </html>
