<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>7. ProSplicer: A Database of Putative Alternative Splicing Information Deriving From Proteins, mRNAs and Expressed Tag Data</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center">  <a href='/recombdb/category01.html'><b>Genomics</b> Overview</a> <a href='/recombdb/category02.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-01-006/'>&lt;&lt;</a> Poster 007 [ <a href='poster-01-007.pdf'>PDF</a> | <a href='poster-01-007.doc'>MSWORD</a> ] <a href='/recombdb/poster-01-008/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>7. ProSplicer: A Database of Putative Alternative Splicing
 Information Deriving From Proteins, mRNAs and Expressed Tag
 Data</h2>
 <p align="center"><b>Hsien-Da Huang</b><a href="#_ftn1" name=
 "_ftnref1" title=""><b>[1]</b></a><b>,</b> <b>Jorng-Tzong
 Horng</b><a href="#_ftn2" name="_ftnref2" title=
 ""><b>[2]</b></a><b>,</b> <b>Chau-Chin Lee</b><a href="#_ftn3"
    name="_ftnref3" title=""><b>[3]</b></a><b>,<br>
  and Baw-Jhiune Liu3</b></p>
 <p><b>Keywords:</b> alternative splicing, database, proteins,
 splice site</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction.</b></p>
 <p>ProSplicer is a database of putative alternative splicing
 information derived by using the alignment of proteins, mRNA
 sequences and ESTs against the human genomic sequences. Proteins,
 mRNA and expressed sequence tags (ESTs) provide valuable evidences
 to reveal the splice variants of genes. Using the alternative
 splicing information in the database can facilitate users to
 investigate the alternative splicing forms and tissue-specific
 expression of the considered genes. ProSplicer is now available at
 <a href=
 "http://bioinfo.csie.ncu.edu.tw/ProSplicer/">http://bioinfo.csie.ncu.edu.tw/ProSplicer/</a>.</p>
 <p>In ProSplicer, whole human genomic sequences and the genes
 including known or novel ones are taken into account to investigate
 the alternative splicing variants. A graphical user interface is
 also provided to reveal the alternative splicing of the considered
 genes. The alternative splicing sites predicted by protein, mRNA or
 EST sequences are also provided. Tissue information provided in
 mRNA, and EST sequences also guide to reveal the tissue favorable
 alternative splicing forms, e.g., exon skipping. The database
 provides keyword search for retrieving and searching the contents
 in the database and graphical interface is also provided to show
 the alternative splicing forms.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Methods</b></p>
 <p>The genomic sequences and genes annotation information are
 obtained from ENSEMBL [1] (Release on May 28, 2002) and there are
 21,786 genes including known and novel genes. The related mRNA
 sequences and EST sequences of the genes are retrieved from UniGene
 [2] (Release 147), containing 96,105 gene clusters of human. The
 EST sequences are from dbEST (Release Feb.22 2002). The protein
 sequences are obtained from the SWISS-PROT and TrEMBL[3] (Release
 20, Mar. 2002).</p>
 <p>The prediction approach of alternative splicing consists of
 three main phases and the system flow is shown in Figure 1. The
 three phases are the preprocessing phase, the alignment phase, and
 the filtering phase. In the preprocessing phase, the gene genomic
 sequences, EST, mRNA sequences and protein sequences, which are
 stored in different biological databases, are collected, converted,
 and integrated into a single database, namely GeneInfo. All the
 sequences are maintained and prepared for the analysis of the
 alignment phase. In the alignment phase, the protein sequences are
 aligned to the gene genomic sequences by TBLASTN [4], as well as
 the mRNA and EST sequences are aligned by the alignment tool, SIM4
 [5]. The exon candidates are generated by both the alignment tools.
 In the filtering phase, we filter the noise of the exon candidates,
 and connect the exon candidates as reasonable transcripts form of
 each EST, mRNA and protein sequences by considering their
 sequential order of matching blocks. Finally, the exon candidates
 of the alternative splicing forms are provided in the database of
 ProSplicer.</p>
 <p align="center"><img border="0" width="370" height="247"
      src="image002.jpg"></p>
 <p align="center">Figure 1: The system flow of ProSplicer.</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Results</b></p>
 <p>As shown in Table 1, the ProSplicer takes the materials of
 21,786 genes form ENSEMBL [5] and totally 2,311,460 sequences
 including protein, mRNA, and EST sequences to investigate the local
 region similarities of sequence to reveal the alternative splicing
 variants. The amount of exon candidates generated by alignment
 tools are shown in Table 1.</p>
 <div align="center">
 <table border="1" cellspacing="0" cellpadding="0">
 <tr>
 <td width="104">
 <p>Sequence Type</p></td>
 <td width="186" colspan="2">
 <p>Amount of Sequences</p></td>
 <td width="194" colspan="2">
 <p>Amount of Exon Candidates</p></td></tr>
 <tr>
 <td width="104" rowspan="2">
 <p>Protein</p></td>
 <td width="75" rowspan="2">
 <p>44,184</p></td>
 <td width="111">
 <p>26,115&nbsp; (human)</p></td>
 <td width="84" rowspan="2">
 <p>442,077</p></td>
 <td width="111">
 <p>279,656&nbsp; (human)</p></td></tr>
 <tr>
 <td width="111">
 <p>18,069&nbsp; (mouse)</p></td>
 <td width="111">
 <p>162,421&nbsp; (mouse)</p></td></tr>
 <tr>
 <td width="104">
 <p>mRNA</p></td>
 <td width="186" colspan="2">
 <p>20,577</p></td>
 <td width="194" colspan="2">
 <p>395,619</p></td></tr>
 <tr>
 <td width="104">
 <p>EST</p></td>
 <td width="186" colspan="2">
 <p>2,246,699</p></td>
 <td width="194" colspan="2">
 <p>12,361,685</p></td></tr>
 <tr>
 <td width="104">
 <p>Total</p></td>
 <td width="186" colspan="2">
 <p>2,311,460</p></td>
 <td width="194" colspan="2">
 <p>13,199,381</p></td></tr></table></div>
 <p align="center">Table 1:&nbsp; The amount of materials and exon
 candidates in ProSplicer.</p>
 <h1>&nbsp;</h1>
 <h1>References</h1>
 <p>[1]&nbsp;&nbsp; Hubbard T, Barker D, Birney E, Cameron G, et al.
 The Ensembl genome database project. <i>Nucl Acids Res</i> 2002,
 30: 38-41.</p>
 <p>[2]&nbsp;&nbsp; Schuler GD, et al: A gene map of the human
 genome. <i>Science</i> 1996, 274 (5287): 540-546.</p>
 <p>[3]&nbsp;&nbsp; Bairoch A, Apweiler R: The SWISS-PROT protein
 sequence database and its supplement TrEMBL in 2000. <i>Nucl Acids
 Res</i> 2000, 28: 45-48.</p>
 <p>[4]&nbsp;&nbsp; Altschul SF, Gish W, Miller W, Myers EW, Lipman
 DJ: Basic local alignment search tool. <i>J Mol Biol</i> 1990, 215:
 403-410.</p>
 <p>[5]&nbsp;&nbsp; Florea L, Hartzell G, Zhang Z, Rubin GM, Miller
 W: A computer program for aligning a cDNA sequence with a genomic
 DNA sequence. <i>Genome Res</i> 1998, 8: 967-974.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p align="left"><a href="#_ftnref1" name="_ftn1" title="">[1]</a>
 Department of Computer Science and Information Engineering,
 National Central University, Taiwan. E&#8209;mail:
 damay@db.csie.ncu.edu.tw</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Department of
 Computer Science and Information Engineering, National Central
 University, Taiwan. E&#8209;mail:
 horng@db.csie.ncu.edu.tw</p></div>
 <div id="ftn3">
 <p><a href="#_ftnref3" name="_ftn3" title="">[3]</a> Department of
 Computer Science and Engineering<sup>,</sup> Yuan-Ze University,
 Taiwan.</p></div></div>
 </body>
 </html>
