<html>

	<head>
		<meta http-equiv="content-type" content="text/html;charset=ISO-8859-1">
		<meta name="generator" content="UNICOM Werbeagentur GmbH | Berlin - Thomas Kusché">
		<title>Welcome to RECOMB 2003</title>
	</head>

	<body bgcolor="#ffffff" leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" vlink="#6699cc" alink="#993333" link="#993333">
		<table border="0" cellpadding="0" cellspacing="0">
			<tr>
				<td valign="top" align="left"><img src="images/banner.gif" width="500" height="80" border="0"></td>
			</tr>
			<tr>
				<td valign="top" align="left">
					<table border="0" cellpadding="10" cellspacing="0">
						<tr>
							<td valign="top" align="left">
								<p class="text"><font face="Arial"><font face="Arial,Helvetica" size="5" color="#993333"><span class="ueberschrift_gross">Accepted Papers 2003<br>
											</span></font><font face="Arial,Helvetica" size="3"><span class="ueberschrift_gross"><br>
											</span></font></font></p>
								<table border="0" cellpadding="0" cellspacing="0">
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Efficient extraction of mapping rules of atoms from enzymatic reaction data.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Tatsuya Akutsu</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>On de-novo interpretation of peptide spectra.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Vineet Bafna and Nathan Edwards</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Haplotypes and informative SNP selection algorithms: don't block out information.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Vineet Bafna, Bjarni V. Halldorsson, Russel Schwartz, Andy Clark and Sorin Istrail</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Modeling dependencies in protein-DNA binding sites.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Yoseph Barash, Gal Elidan, Nir Friedman and Tommy Kaplan</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Efficient exact p-value computation and applications to biosequence analysis.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Gill Bejerano</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Towards optimally multiplexed applications of universal DNA tag systems.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Amir Ben-Dor, Tzvika Hartman, Benno Schwikowski, Roded Sharan and Zohar Yakhini</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>A comparative analysis method for detecting binding sites in coding regions.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Mathieu Blanchette</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Designing seeds for similarity search in genomic DNA.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Jeremy Buhler, Uri Keich and Yanni Sun</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Discovery of conserved sequence pair templates associated with gene expression differences in yeast.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Derek Y. Chiang, Alan M. Moses, Manolis Kamvysselis, Eric S. Lander and Michael B. Eisen</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Maximum likelihood on four taxa phylogenetic trees: analytic solutions.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Benny Chor, Amit Khetan and Sagi Snir</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Integrated probabilistic model for functional prediction of proteins.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Minghua Deng, Ting Chen and Fengzhu Sun</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Large scale reconstruction of haplotypes from genotype data.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Eleazar Eskin, Eran Halperin and Richard M. Karp</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Optimizing exact genetic linkage computations.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Ma'ayan Fishelson and Dan Geiger</i><i><br>
													<br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Finding recurrent sources in sequences.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Aristides Gionis and Heikki Mannila</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Model-based inference of haplotype block variation.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Gideon Greenspan and Dan Geiger</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Accurate detection of very sparse sequence motifs.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Andreas Heger, Michael Lappe and Liisa Holm</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Engineering a scalable placement heuristic for DNA probe arrays.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>A. B. Kahng, I. I. Mandoiu, P. A. Pevzner, S. Reda and A. Z. Zelikovsky</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Whole-genome comparative annotation and regulatory motif discovery in multiple yeast species.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Manolis Kamvysselis, Nick Patterson, Bruce Birren, Bonnie Berger and Eric Lander</i><i><br>
												</i></font>
											<p></p>
										</td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Joint classifier and feature optimization for cancer diagnosis using gene expression data.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Balaji Krishnapuram, Lawrence Carin and Alexander Hartemink</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>A polynomial-time nuclear vector replacement algorithm for automated NMR resonance assignments.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Christopher James Langmead, Anthony K. Yan, Ryan Lilien, Lincong Wang and Bruce Randall Donald</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>A complete and effective move set for simplified protein folding.<br>
												</b></font><font face="Arial" size="3" color="#993333">N. Lesh, M. Mitzenmacher and S. Whitesides<br>
											</font><font face="Arial" size="3"><br>
											</font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Gene selection criterion for discriminant microarray data analysis based on extreme value distributions.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Wentian Li and Ivo Grosse</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Efficient rule-based haplotyping algorithms for pedigree data.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Jing Li and Tao Jiang</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Haplotype reconstruction from SNP alignment.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Lei Li, Jong Hyun Kim and Michael S. Waterman</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>A multi-expert system for the automatic detection of protein domains from sequence information.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Niranjan Nagarajan and Golan Yona</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Resolution of haplotypes and haplotype frequencies from SNP genotypes of pooled samples.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Itsik Pe'er and Jacques S. Beckmann</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Transforming men into mice: the Nadeau-Taylor chromosomal breakage model revisited.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Pavel Pevzner and Glenn Tesler</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Score functions for assessing conservation in locally aligned regions of DNA from two species.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Krishna M. Roskin, Mark Diekhans and David Haussler</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Approximation of protein structure for fast similarity measures.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Fabian Schwarzer and Itay Lotan</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Combining phylogenetic and hidden Markov models in biosequence analysis.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Adam Siepel and David Haussler</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Using a mixture of probabilistic decision trees for direct prediction of protein function.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Umar Syed and Golan Yona</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Modeling transcription programs: inferring binding site activity and dose-response model optimization.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Amos Tanay and Ron Shamir</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Physical network models and multi-source data integration.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Chen-Hsiang Yeang and Tommi Jaakkola</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Maximum entropy modeling of short sequence motifs with applications to RNA splicing signals.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Gene Yeo and Christopher Burge</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
									<tr>
										<td width="15" valign="top" align="left"><font face="Arial" size="3">&#149;</font></td>
										<td valign="top" align="left"><font face="Arial" size="3"><b>Dynamic programming algorithms for haplotype block partitioning: applications to human chromosome 21 haplotype data.<br>
												</b></font><font face="Arial" size="3" color="#993333"><i>Kui Zhang, Fengzhu Sun, Michael S. Waterman and Ting Chen</i><i><br>
												</i></font><font face="Arial" size="3"><i><br>
												</i></font></td>
									</tr>
								</table>
								<br>
								<font face="Arial" size="3"><br>
								</font></td>
						</tr>
					</table>
				</td>
			</tr>
		</table>
	</body>

</html>