<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 
 <html>
 <head>
 
 <title>87. An Automated Analysis of Almost All Available ABC Proteins</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category03.html'>&lt;&lt;</a> <a href='/recombdb/category04.html'><b>Molecular Evolution</b> Overview</a> <a href='/recombdb/category05.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-04-086/'>&lt;&lt;</a> Poster 087 [ <a href='poster-04-087.pdf'>PDF</a> | <a href='poster-04-087.tgz'>TeX</a> | <a href='poster-04-087.ps'>PostScript</a> ] <a href='/recombdb/poster-05-088/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>87. An Automated Analysis of
 Almost All Available ABC Proteins<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font></div>
 <div align="center"><b>Michael Spitzer, <a name="tex2html1"
    href="#foot54"><sup>1</sup></a> Stefan Lorkowski, <a name=
    "tex2html2" href="#foot55"><sup>2</sup></a> Paul Cullen,
    <a name="tex2html3" href="#foot56"><sup>3</sup></a> Georg
    Fuellen <a name="tex2html4" href=
    "#foot57"><sup>4</sup></a></b></div>
 <p><font size="-1"><br>
 <br></font></p>
 <p><font size="-1"><b>Keywords:</b> ABC proteins, homology search,
 phylogeny</font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Abstract</font></a></h1>
 <font size="-1">RiPE (Rigorous Phylogeny Environment) is an
 automatic procedure to perform evolutionary analyses of protein
 families using only homologous protein sequence data. Using RiPE,
 we assembled a set of 1,000 ABC protein sequences from model
 organisms, and subjected these to phylogenetic analyses.<br>
  We searched the proteomes of <i>Homo</i>, <i>Drosophila</i>,
 <i>Arabidopsis</i>, <i>Encephalitozoon</i>, <i>Saccharomyces</i>,
 <i>Mus</i>, <i>Rattus</i>, <i>Fugu</i>, <i>Danio</i>,
 <i>Caenorhabditis</i>, <i>Anopheles</i>,
 <i>Schizosaccharomyces</i>, <i>Oryza</i>, <i>Guillardia</i>, the
 archaeon <i>Methanococcus jannaschii</i> and three representative
 bacteria (<i>Escherichia coli</i>, <i>Synechocystis sp.</i>,
 <i>Mycoplasma genitalium</i>).<br>
  The multiple alignment was done using ClustalW [<a href=
 "#CLUSTALW">1</a>]; the overall tree was computed
 by neighbor joining via "quicktree" [<a href=
 "#QUICKTREE">2</a>]. In the resulting
 phylogenetic tree, sequences known to belong to the same subfamily
 were placed into the same subtree, even if bootstrap values were
 very low. More specifically, taking the human subfamily
 designations, we can reproduce classifications from the literature
 in the case of <i>Drosophila</i>, <i>Arabidopsis</i> and
 <i>Encephalitozoon</i>. This reproduction was correct in almost all
 cases. In turn, we can place high confidence in the subfamily
 classification labels that we inferred for proteins of model
 organisms for which no classification has&nbsp; yet been published
 as of today. Based on these phylogenetic analyses, we discuss
 possible naming schemes for ABC protein inventories of the model
 organisms that have not been classified so far.</font> 
 <p></p>
 <h2><a name="SECTION00020000000000000000">Bibliography</a></h2>
 <dl compact>
 <dt><a name="CLUSTALW">1</a></dt>
 <dd>Thompson, J.D., Higgins, D.G. and Gibson, T.J. 1994. CLUSTAL W:
 improving the sensitivity of progressive multiple sequence
 alignment through sequence weighting, positions-specific gap
 penalties and weight matrix choice. <em>Nucleic Acids Research</em>
 22:4673-4680</dd>
 <dt><a name="QUICKTREE">2</a></dt>
 <dd>Howe K., Bateman A. and Durbin R. 2002. QuickTree: building
 huge Neighbour-Joining trees of protein sequences.
 <em>Bioinformatics</em> 18(11):1546-1547</dd></dl>
  
 <hr>
 <h4>Footnotes</h4>
 <dl>
 <dt><a name="foot54">... Spitzer,</a><a href=
 "#tex2html1"><sup>1</sup></a></dt>
 <dd>Integrated Functional Genomics, IZKF, University Hospital
 Muenster (Hautklinik), Von-Esmarch-Str. 56, 48149 Muenster,
 Germany. E-mail: 
 <div align="center">
 <b><tt>michael.spitzer@uni-muenster.de</tt></b></div></dd>
 <dt><a name="foot55">... Lorkowski,</a><a href=
 "#tex2html2"><sup>2</sup></a></dt>
 <dd>MAFAPS Research Group, University Hospital Muenster (IfA),
 Domagkstrasse 3, 48149 Muenster, Germany. E-mail: 
 <div align="center">
 <b><tt>stefan.lorkowski@uni-muenster.de</tt></b></div></dd>
 <dt><a name="foot56">... Cullen,</a><a href=
 "#tex2html3"><sup>3</sup></a></dt>
 <dd>Ogham GmbH, Mendelstrasse 11, 48149 Muenster, Germany. E-mail: 
 <div align="center"><b><tt>cullen@ogham.de</tt></b></div></dd>
 <dt><a name="foot57">... Fuellen</a><a href=
 "#tex2html4"><sup>4</sup></a></dt>
 <dd>Integrated Functional Genomics, IZKF, University Hospital
 Muenster (Hautklinik), Von-Esmarch-Str. 56, 48149 Muenster,
 Germany. E-mail: 
 <div align="center">
 <b><tt>fuellen@uni-muenster.de</tt></b></div></dd></dl>
 <br>
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
