<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>77. Classification of species based on DNA style</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category03.html'>&lt;&lt;</a> <a href='/recombdb/category04.html'><b>Molecular Evolution</b> Overview</a> <a href='/recombdb/category05.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-04-076/'>&lt;&lt;</a> Poster 077 [ <a href='poster-04-077.pdf'>PDF</a> | <a href='poster-04-077.doc'>MSWORD</a> ] <a href='/recombdb/poster-04-079/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>77. Classification of species based on DNA style</h2>
 <p align="center"><b>Charles Chapus<a href="#_ftn1" name="_ftnref1"
 title="">[1]</a>, Bernard Fertil<sup>1</sup>, Scott
    Edwards<a href="#_ftn2" name="_ftnref2" title="">[2]</a>, Alain
    Giron<sup>1</sup>, Patrick&nbsp;Deschavanne1</b></p>
 <p><b>Keywords:</b> genome, genomic signature, species
 classification, phylogeny</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction</b></p>
 <p>The recent availability of complete genomes opens a new field of
 research devoted to the general analysis of their global structure
 without regard to gene interpretation. The exploration of such
 large sequences requires a new kind of data analysis. We have shown
 that the whole set of oligonucleotide frequencies of a sequence is
 species specific [1]. DNA fragments conserved the signature of the
 species they come from [2] and such analysis may therefore be
 considered as a genomic signature. As the signature is very species
 specific, it can be used to classify species on its mere basis. The
 distance between images may quantify phylogenetic proximity. In
 this poster we will first search for the minimal length of DNA
 which could be used as reference for species signature. In a second
 step, we will classify species on a taxonomic basis and establish
 phylogenies based on DNA style.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Methods and Results</b></p>
 <p>Seventy-four complete genomes belonging to the three domains of
 life were investigated. Comparison of DNA fragment signatures to
 that of the whole genome showed that fragments as short as 1 kb
 conserved the species signature. This conservation is very good for
 prokaryotes and of less quality for eukaryotes due to intrinsic
 intra-genomic variations. However, when using partial genome
 sequence as reference, two variability factors are added, that from
 the fragment to classify and that from the one representing the
 species. Analysis of the classification result as a function of the
 two parameters (length of tested fragment and length of reference
 fragment) have shown that for reference sequence representing at
 least 10 % of the genome, the classification quality is quite
 similar to that obtained with the complete genome as reference. For
 reference fragment of 5 Kb, about 75% of 2.5 kb test fragments are
 correctly classified. This supports the previous hypothesis of a
 species-specific DNA style and allows to compare genomic DNA
 fragments of different size found in databases for species
 classification.</p>
 <p>We established the signature of about 10 000 species of the 3
 domains of life with the following criterion:</p>
 <p>1/ DNA must be genomic (no mRNA, mitochondrial or chloroplast
 sequences)</p>
 <p>2/ 1.5 kb as minimum size for DNA sequence incorporated for
 establishing species signature</p>
 <p>The resulting size of DNA sequences classified varies from 1.5
 kb to 32 MB. In a first step, a leave-one-out discriminant
 classification was done in descending order of taxonomic complexity
 (fig 1). Signature distinguishes very well between prokaryotes and
 eukaryotes as well as bacteria from archea. Within the eukarya
 domain, the separation is of lesser quality but with rather good
 results (fig 1). This allows proposing the use sequence signature
 to explore phylogenetic relationships. We have already show that
 very good phylogenetic trees were obtained when using homologous
 sequence signature[3]. We study the place of flightless birds
 (ratites) in the bird tree. With using non-homologous sequences of
 different sizes, varying from 1.5 to 600 kb, we have shown that the
 traditional hypothesis of basal ratites and monophyly of
 non-ratites birds (neognaths) was strongly supported by our
 analysis. This encouraging result leads us to explore the phylogeny
 of prokaryotes using complete genome signatures.</p>
 <p align="center">Figure 1: classification of 10 000 species based
 on their sequence signature. The percentage of good classified
 species in the corresponding class is given as a function of
 taxonomic families.</p>
 <p><img width="493" height="264" src="image003.gif"><br clear=
 "all">
  &nbsp;</p>
 <br clear="all">
  
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>References</b></p>
 <p>[1] Deschavanne, P.J., A. Giron, J. Vilain, G. Fagot, and B.
 Fertil. 1999. Genomic signature: characterization and
 classification of species assessed by Chaos Game Representation of
 sequences<i>.</i> <i>Molecular Biology and Evolution</i>.
 16:1391&ndash;1399.</p>
 <p>[2] Deschavanne, P., A. Giron, J. Vilain, C. Dufraigne, and B.
 Fertil. 2000. Genomic signature is preserved in short DNA
 fragments. In: <i>BIBE2000 IEEE international Symposium on
 bio-informatics &amp; biomedical engineering</i>, Washington USA.
 pp.161-167.</p>
 <p>[3] Chapus C., Fertil B., Giron A.&nbsp; and P. Deschavanne.
 2002. Genomic signature: a global sequence analysis concept applied
 to phylogeny. In: <i>The sixth Annual International Conference on
 Computational Molecular Biology</i> (<i>RECOMB2002).</i>
 Washington, USA. pp.34-35</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a> INSERM - U 494
 - CHU Piti&eacute;-Salp&ecirc;tri&egrave;re, Paris, France.
 E&#8209;mail: chapus@imed.jussieu.fr</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Dept of
 Zoology and Burke Museum, University of Washington, Seatlle,
 Washington 98195, USA</p></div></div>
 </body>
 </html>
