<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 
 <html>
 <head>
 
 <title>140. Discriminative Clustering Analysis</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category05.html'>&lt;&lt;</a> <a href='/recombdb/category06.html'><b>Gene Expression</b> Overview</a> <a href='/recombdb/category07.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-06-139/'>&lt;&lt;</a> Poster 140 [ <a href='poster-06-140.pdf'>PDF</a> | <a href='poster-06-140.tgz'>TeX</a> | <a href='poster-06-140.ps'>PostScript</a> ] <a href='/recombdb/poster-06-141/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>140. Discriminative
 Clustering Analysis<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font></div>
 <div align="center"><b>Francisco J. Molina <a name="tex2html1"
    href="#foot26"><sup>1</sup></a></b></div>
 <p><font size="-1"><br>
 <br></font></p>
 <p><font size="-1"><b>Keywords:</b> gene expression, microarray,
 clustering</font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Introduction</font></a></h1>
  <font size="-1">To analyze microarray expression data, we
 introduce a clustering algorithm that finds groups of genes with
 similar expression patterns. The found clusters are ranked
 according to their capability to discriminate between two or more
 kinds of samples. Instead of clustering based on only the
 similarity between expression profiles, the algorithm takes into
 account a classification of the samples in disjoint categories.
 This classification is based upon one or more experimental
 conditions under which the samples were taken. These conditions
 could be extracelular environments, i.g., ``in vivo'' and ``in
 vitro'', or different types of breast cancer, i.g., ``sporadic'',
 ``brac1'' and ``brac2''. By using the label of the microarray
 samples, we provide extra information to cluster genes.<br></font> 
 <p><font size="-1">The algorithm iteratively extracts the most
 discriminative cluster within the remaining genes. The
 discriminative capability of a cluster quantifies how much a
 cluster discriminates between sample classes, that is, how
 differently the genes in that cluster are expressed across sample
 classes.<br></font></p>
 <p><font size="-1">To fix ideas, let us suppose that several
 microarray experiments are performed and that we are given certain
 cluster <img width="11" height="13" align="bottom" border="0"
      src="img1.png" alt="$ c $"> of genes used in those
      experiments.<br></font></p>
 <p><font size="-1">To compute the discriminative capability of
 <img width="11" height="13" align="bottom" border="0" src=
 "img1.png" alt="$ c $">, we assign a probability density to every
 sample class and then we quantify the overlapping between those
 probability densities. The bigger is the overlapping, the less
 <img width="11" height="13" align="bottom" border="0" src=
 "img1.png" alt="$ c $"> discriminates across sample classes. When
 this overlapping is large, for a wide range of expression levels,
 the infered probabilities of observing those expression levels in
 different sample classes is similar. This makes more difficult to
 distinguish between those sample classes by considering only the
 expression data from the genes in cluster <img width="11"
      height="13" align="bottom" border="0" src="img1.png" alt=
      "$ c $">.<br></font></p>
 <p><font size="-1">Once identified a cluster of genes
      <img width="12" height="14" align="bottom" border="0" src=
      "img2.png" alt="$ d $"> that is very discriminative, we can
      label a new sample by assigning it to the sample class that
      maximizes the likelihood of the new sample's expression
      data.</font></p>
 <p><font size="-1">The more discriminative the cluster
      <img width="12" height="14" align="bottom" border="0" src=
      "img2.png" alt="$ d $"> is, the smaller is the probability of
      missclassifying the new sample.</font></p>
 <p></p>
 <h2><a name="SECTION00020000000000000000">Bibliography</a></h2>
 <dl compact>
 <dt><a name="HB98">1</a></dt>
 <dd>Dettling, M. and Buhlmann, P. 2002. Supervised clustering of
 genes. <em>Genome Biology</em></dd></dl>
  
 <hr>
 <h4>Footnotes</h4>
 <dl>
 <dt><a name="foot26">... Molina</a><a href=
 "#tex2html1"><sup>1</sup></a></dt>
 <dd>Lawerence Berkeley National Laboratory, Department of Energy,
 USA E-mail: 
 <div align="center">
 <b><tt>fjmolina@lbl.gov</tt></b></div></dd></dl>
 <br>
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
