<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 
 <html>
 <head>
 
 <title>145. Estimating the score distributions of induced and non-induced genes in microarray experiments</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category05.html'>&lt;&lt;</a> <a href='/recombdb/category06.html'><b>Gene Expression</b> Overview</a> <a href='/recombdb/category07.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-06-144/'>&lt;&lt;</a> Poster 145 [ <a href='poster-06-145.pdf'>PDF</a> | <a href='poster-06-145.tgz'>TeX</a> | <a href='poster-06-145.ps'>PostScript</a> ] <a href='/recombdb/poster-06-146/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>145. Estimating the score
 distributions of induced and non-induced genes in microarray
 experiments<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font></div>
 <div align="center"><b>Stefanie Scheid <a name="tex2html1" href=
 "#foot41"><sup>1</sup></a> and Rainer Spang <a name="tex2html2"
 href="#foot41"><sup>1</sup></a></b></div>
 <p><font size="-1"><br>
 <br></font></p>
 <p><font size="-1"><b>Keywords:</b> score distribution, mixture,
 induced genes, False Discovery Rate</font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Introduction</font></a></h1>
 <p><font size="-1">A microarray experiment measures simultaneously
 the abundance of mRNA for several thousands of genes in a given
 tissue. With two classes of tissue or state of patient one can
 search for genes that show differential gene expression with
 respect to the classes. Based on the microarray data we assign a
 Wilcoxon ranksum score for differential gene expression to every
 gene. Typically a high score corresponds to overexpression and a
 low score to underexpression. A gene showing differential gene
 expression is called ``induced''.</font></p>
 <p><font size="-1">The score distributions of microarray
 experiments are mixtures of score distributions of induced and
 non-induced genes. The two distributions can be distinguished if
 the truly induced genes show very high (or low resp.) scores.
 However, in a typical setting the two distributions overlap and
 only highly induced genes can be selected with common methods. The
 main part of the induced genes merge with non-induced genes in
 ``twilight zones''. It is a challenging problem to discover where
 twilight zones are and how many truly induced genes one can expect
 at a certain score level. If we observe a score for a specific gene
 we cannot only tell how deep it is situated in a twilight zone but
 also how likely it is to be an induced gene if the mixture is known
 at that score level.</font></p>
 <h1><a name="SECTION00020000000000000000"><font size=
 "+1">Method</font></a></h1>
 <p><font size="-1">We introduce a new method that is based on the
 idea of <i>minimal</i> removal of genes such that the score
 distribution of remaining genes is similar to a score distribution
 of non-induced genes. The original score distribution is derived
 from two-class data. Distributions for non-induced genes are
 generated by class permutation. We divide the range of scores into
 intervals (bins) and count the relative bin frequencies of the
 permuted samples. For each bin a lower and upper quantile of
 relative bin frequency is calculated.</font></p>
 <p><font size="-1">The basic idea is to remove genes until all
 relative bin frequencies of the remaining original scores are
 contained within the quantile intervals of the permutations. The
 remaining original scores therefore appear as being derived from
 non-induced genes. We estimate the two mixtures as the relative bin
 frequencies of the remaining scores for the non-induced and of the
 excluded scores for the induced part. Removing induced genes makes
 the two distributions more similar. However, removing non-induced
 genes in addition does not necessarily lead to more dissimilar
 distributions. Therefore, the central point is to exclude as few
 scores as possible in order to provide a conservative estimate for
 the number of induced genes.</font></p>
 <p><font size="-1">To take advantage of the permutations' diversity
 we compare the original scores to each sample of permutation
 scores. In a first step the differences of their absolute bin
 frequencies are computed. One by one we exclude those genes whose
 original <em>and</em> permuted scores are redundant with respect to
 the differences. This selection stops if either all differences are
 zero or if no more suitable genes can be found.</font></p>
 <p><font size="-1">In a second step we compare the relative bin
 frequencies of the remaining original scores to the quantile
 intervals and remove genes if necessary. If a bin contains a number
 too small to reach the lower quantile, excluded genes are included
 again. This procedure stops if either all relative bin frequencies
 fit into the quantile intervals or if no more suitable genes can be
 found.</font></p>
 <p><font size="-1">We conduct the method with a high number of
 permutations and estimate the score distribution of non-induced
 genes as the mean relative bin frequencies of the remaining
 original scores over all permutations. In a simulation study we
 evaluate the new method under several settings of sparsity. The
 choice of the lower and upper quantile points and the number and
 design of the bin segmentation is a topic for further
 investigation.</font></p>
 <h1><a name="SECTION00030000000000000000"><font size="+1">Context
 to established approaches</font></a></h1>
 <p><font size="-1">Classical approaches of separation like
 estimating mixture models require knowledge of underlying
 distribution functions. A method that does not need this
 information is a variant of the recently introduced False Discovery
 Rate (FDR). In a multiple testing situation it estimates the
 proportion of genes that are falsely called induced among all genes
 called induced ([<a href="#BH95">1</a>],
    [<a href="#S01">2</a>]). Used in the common
    way the FDR only touches the twilight zone because genes are
    called induced whose scores exceed or fall below a certain
    threshold.</font></p>
 <p><font size="-1">If we define the FDR in an altered way and call
 all genes induced that fall into a specific bin, we can calculate a
 <em>bin-wise FDR</em> to estimate the score distribution of
 non-induced genes. The disadvantage of this method is the need of
 prior knowledge, i.e. an adequate estimate for the overall amount
 of non-induced genes. The commonly used estimator for the normal
 FDR as given in [<a href="#TTC01">3</a>] will
 overestimate the amount of non-induced genes if the scores of
 induced genes are hidden among scores of non-induced
 genes.</font></p>
 <p></p>
 <h2><a name="SECTION00040000000000000000">Bibliography</a></h2>
 <dl compact>
 <dt><a name="BH95">1</a></dt>
 <dd>Benjamini, Y., Hochberg, Y. 1995. Controlling the False
 Discovery Rate: a Practical and Powerful Approach to Multiple
 Testing. <em>Journal of the Royal Statistical Society B</em>,
 57(1):289-300.</dd>
 <dt><a name="S01">2</a></dt>
 <dd>Storey, J.&nbsp;D. 2001. <em>The positive False Discovery Rate:
 A Bayesian Interpretation and the q-value</em>, Stanford
 University: Stanford Technical Report.</dd>
 <dt><a name="TTC01">3</a></dt>
 <dd>Tusher, V.&nbsp;G., Tibshirani, R., Chu, G. 2001. Significance
 analysis of microarrays applied to the ionizing radiation response.
 <em>Proceedings of the National Academy of Sciences</em>,
 98(9):5116-5121.</dd></dl>
  
 <hr>
 <h4>Footnotes</h4>
 <dl>
 <dt><a name="foot41">... Scheid</a><a href=
 "#tex2html1"><sup>1</sup></a></dt>
 <dd>Computational Diagnostics Group, Department of Computational
 Molecular Biology, Max-Planck-Institute for Molecular Genetics,
 Ihnestrasse 63-73, D-14195 Berlin, Germany. E-mail: 
 <div align="center">
 <b><tt>stefanie.scheid@molgen.mpg.de</tt>,</b></div>
 <div align="center">
 <b><tt>rainer.spang@molgen.mpg.de</tt></b></div></dd></dl>
 <br>
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
