<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>86. Expanding protein universe and its origin from the biological big bang</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category03.html'>&lt;&lt;</a> <a href='/recombdb/category04.html'><b>Molecular Evolution</b> Overview</a> <a href='/recombdb/category05.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-04-085/'>&lt;&lt;</a> Poster 086 [ <a href='poster-04-086.pdf'>PDF</a> | <a href='poster-04-086.doc'>MSWORD</a> ] <a href='/recombdb/poster-04-087/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>86.&nbsp; Expanding protein universe and its origin from the
 biological big bang</h2>
 <p align="center"><b>Boris Shakhnovich<a href="#_ftn1" name=
 "_ftnref1" title="">[1]</a>, Nikolay V. Dokholyan<a href="#_ftn2"
 name="_ftnref2" title="">[2]</a>, and Eugene I.
    Shakhnovich<a href="#_ftn3" name="_ftnref3" title=
    "">[3]</a></b></p>
 <p><b>Keywords:</b> molecular evolution, structure comparison,
 power law behavior</p>
 <p><b>&nbsp;</b></p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp; Introduction</b></p>
 <p>The bottom-up approach to understanding the evolution of
 organisms is by studying molecular evolution. With the large number
 of protein structures identified in the past decades, we have
 discovered peculiar patterns that nature imprints on protein
 structural space in the course of evolution. In particular, we have
 discovered that the universe of protein structures is organized
 hierarchically into a scale-free network. By understanding the
 cause of these patterns, we attempt to glance at the very origin of
 life.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</b>
 <b>Methods and results</b></p>
 <p>We study the properties of the largest cluster of the Protein
 domain universe graph (PDUG) as a function of cutoff similarity
 score Zmin. For control, we compare the properties of the largest
 PDUG cluster to that of random graphs that have the same number of
 nodes and edges as PDUG and the same distribution of pairwise
 Z-scores, but where edges are connected randomly. First, we compute
 the size of a largest cluster as a function of Zmin. We find the
 sharp transition of the size of the largest cluster at Zmin = Zc ~
 9 Fig 2. Random graphs feature similar transition, but at a higher
 value of Zmin ~ 11 Fig2. The distribution of cluster sizes depends
 significantly on whether Zmin &gt; Zc or Zmin &lt; Zc for PDUG and
 for random graphs. We also find that the probability distribution
 P(M) of cluster sizes M for both PDUG and random graphs follows a
 power-law at their respective Z_c: P(M) ~ M<sup>2.5</sup>. The
 observed power-law behavior of P(M) is, however, a consequence of
 criticality at Z_c as it is featured prominently both for PDUG and
 random clusters. It is a generic percolation phenomenon that has
 been observed and explained in percolation and in the random graph
 theories.&nbsp;</p>
 <p>In order to characterize the structural properties of the PDUG
 we measure the probability distribution P(k) of the number of edges
 per node k taken at Zmin=Zc for individual clusters. It is known
 that P(k) distinguishes random graphs from various graphs observed
 in science and technology. Specifically, many known graphs, such as
 WWW and scientific collaboration networks are scale-free: P(k) ~
 k<sup>a</sup>. In a number of studied cases a~ 3. In drastic
 contrast with equivalent random graph, PDUG is scale-free with
 P(k)~k<sup>-1.6</sup> with high degree of statistical
 significance</p>
 <p>p-value&lt;10<sup>-8</sup>&nbsp; Fig 3 a). As expected, PDUG is
 a scale-free graph only at Zmin~Z_c, while above and below Zc PDUGs
 are trivial graphs (one large cluster and a few orphans below Z_c
 and mostly orphans above Zc. Importantly, the random graph is not
 scale-free at any value of Zmin (Fig 3b).</p>
 <p>
 <b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</b>
 Our new rigorous method of clustering protein <i>structures</i>
 provides a number of insights. Firstly, we discovered that PDUG is
 by far non-random, but rather represents a scale-free network.
 Secondly, we show that such a unique property can be quantitatively
 explained from a divergent evolution prospective, giving support to
 the view that all proteins might have evolved from a few
 precursors. It follows that the observed non-uniform distribution
 of protein folds is a ``random'' consequence of the genesis of
 proteins by a divergent evolution mechanism, rather than the
 unequal designability of protein folds.</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp; Figures and tables</b></p>
 <p><b>Fig 1</b> An example of large cluster of TIM-barrel fold
 protein domains.&nbsp; Protein domains whose DALI similarity
 Z-score is greater than Z<sub>min</sub> = 9 are connected by
 lines.</p>
 <p><img width="288" height="144" src="image004.jpg"></p>
 <p><b>&nbsp;</b></p>
 <p><b>Fig 2.</b> The dependence of the number of proteins in the
 maximal cluster on the threshold value of Z-score for PDUG and
 random graphs.&nbsp;&nbsp;</p>
 <p><img width="456" height="187" src="image006.jpg"></p>
 <h1>Fig 3. Distribution of node connectivity for PDUG (a) for
 random graph (b). Node connectivity denotes how many proteins is a
 given protein structurally similar to.</h1>
 <h1>&nbsp;</h1>
 <h1>References</h1>
 <p>Barabasi, A. L. and R. Albert (1999). "Emergence of scaling in
 random networks." <u>Science</u> <b>286</b>(5439): 509-12.</p>
 <p>Bollobas, B. (1985). <u>Random Graphs</u>. London, Academic
 Press.</p>
 <p>Dietmann, S., J. Park, et al. (2001). "A fully automatic
 evolutionary classification of protein folds: Dali Domain
 Dictionary version 3." <u>Nucleic Acids Res</u> <b>29</b>(1):
 55-7.</p>
 <p>Holm, L. and C. Sander (1993). "Protein structure comparison by
 alignment of distance matrices." <u>J Mol Biol</u> <b>233</b>(1):
 123-38.</p>
 <p>Qian, J., N. M. Luscombe, et al. (2001). "Protein family and
 fold occurrence in genomes: power-law behaviour and evolutionary
 model." <u>J Mol Biol</u> <b>313</b>(4): 673-81.</p>
 <p>Todd, A. E., C. A. Orengo, et al. (2001). "Evolution of function
 in protein superfamilies, from a structural perspective." <u>J Mol
 Biol</u> <b>307</b>(4): 1113-43.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a>&nbsp;
 Bioinformatics Department, Boston University, Boston, MA 02215,
 USA, E-mail: borya@acs.bu.edu</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title=
 "">[2]</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Department of
 Biochemistry and Biophysics, University of North Caroline at Chapel
 Hill School of Medicine, Chapel Hill, NC 27599, USA E&#8209;mail:
 dokh@med.unc.edu</p></div>
 <div id="ftn3">
 <p><a href="#_ftnref3" name="_ftn3" title=
 "">[3]</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Department of
 Chemistry and Chemical Biology, Harvard University,&nbsp;
 Cambridge, Ma 02138, USA E-mail:
 Eugene@belok.harvard.edu</p></div></div>
 </body>
 </html>
