<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 
 <html>
 <head>
 
 <title>5. Improving SNP genotyping results by introducing Gaussian mixture models for genotype calling</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center">  <a href='/recombdb/category01.html'><b>Genomics</b> Overview</a> <a href='/recombdb/category02.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-01-004/'>&lt;&lt;</a> Poster 005 [ <a href='poster-01-005.pdf'>PDF</a> | <a href='poster-01-005.tgz'>TeX</a> | <a href='poster-01-005.ps'>PostScript</a> ] <a href='/recombdb/poster-01-006/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>5. Improving SNP genotyping
 results by introducing Gaussian mixture models for genotype
 calling<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font></div>
 <div align="center"><b>Christian Gieger, <a name="tex2html1"
    href="#foot97"><sup>1</sup></a> Maik Kschischo, <a name=
    "tex2html2" href="#foot98"><sup>2</sup></a> Rainer Kern,
    <a name="tex2html3" href="#foot99"><sup>3</sup></a> Martin
    Steinhauser, <a name="tex2html4" href=
    "#foot100"><sup>4</sup></a> Ralf Tolle <a name="tex2html5" href=
    "#foot101"><sup>5</sup></a></b></div>
 <p><font size="-1"><br>
 <br></font></p>
 <p><font size="-1"><b>Keywords:</b> SNP, clustering, Gaussian
 mixture model, genotyping, quality control</font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Introduction.</font></a></h1>
   
 <p><font size="-1">Genetic diversity and its functional relevance
 is a main topic of current genomic research. Single nucleotide
 polymorphisms (SNPs) are used in genetic mapping, target gene
 association studies and in probing the causes of differential
 responses to drug treatment. SNP genotyping on an industrial scale
 relies on a few technologies, which are robust, amenable to
 automation and relatively cost-efficient. One of the genotyping
 assays which has been adopted in numerous labs is the
 template-directed dye-terminator incorporation assay with
 fluorescence polarization detection (FP-TDI) [<a href=
 "#CLK99">2</a>]. Here a genotyping primer is
 annealed next to the prospective polymorphic site and extended by
 the respective dye-terminator labeled base(s). Which dye-terminator
 was incorporated is then determined by measuring the decrease in
 fluorescence depolarization for the excitation/emission wavelengths
 corresponding to the fluorescent dyes used for labeling.</font></p>
 <p><font size="-1">Raw data are analyzed by plotting the data for
 the two colors which are representing the two alleles of a SNP.
 Ideally, this XY-plot leads to distinct clusters representing the
 different genotypes. There are however cases, when data points
 cannot be easily assigned to an individual cluster [<a href=
 "#GSN02">4</a>]. Our approach is to apply
 Gaussian mixture (GM) models [<a href=
 "#BR93">1</a>] to the clustering of SNP assay
 data in order to reduce human intervention in genotype calling to a
 minimum and at the same time to provide objective and quantitative
 criteria for the assessment of assay quality. We derived several
 quantities in the framework of Gaussian mixture models which
 provide figures of merit for the assignment of observations to
 clusters, thus vitally improving the whole process of primer
 extension SNP genotyping and opening a door to fully automated SNP
 genotyping on an industrial scale.</font></p>
 <h1><a name="SECTION00020000000000000000"><font size=
 "+1">Methods.</font></a></h1>
   
 <p><font size="-1">A Gaussian mixture model assumes that each group
 (genotype cluster) of the data is generated by an underlying
 probability distribution. Let 
 <img width="115" height="32" align="middle" border="0" src=
 "img1.png" alt="$\bf y = \{\bf y_1,...,\bf y_n\}$"> denote the
 independent multivariate observations to be clustered, with
 <img width="19" height="29" align="middle" border="0" src=
 "img2.png" alt="${\bf y_i}$"> being a <img width="12" height="14"
 align="bottom" border="0" src="img3.png" alt="$d$">-dimensional
 vector (<img width="76" height="29" align="middle" border="0"
      src="img4.png" alt="$i=1,...,n$">). In our case we have
      <img width="41" height="14" align="bottom" border="0" src=
      "img5.png" alt="$d=2$"> and the components of <img width="19"
      height="29" align="middle" border="0" src="img2.png" alt=
      "${\bf y_i}$"> correspond to the scaled values of the FP-TDI.
      Let <img width="19" height="14" align="bottom" border="0" src=
      "img6.png" alt="$K$"> be the number of components in the data.
      Under the assumtion of independent and identical distributed
      observations <img width="19" height="29" align="middle"
      border="0" src="img2.png" alt="${\bf y_i}$"> one obtains for
      the likelihood <img width="15" height="14" align="bottom"
      border="0" src="img7.png" alt="${\cal L}$"> of the mixture
      model:</font></p>
 <div align="right">
 <table width="100%" align="center">
 <tr valign="middle">
 <td align="center" nowrap><a name=
      "Eq.likelihood"></a><img width="316" height="56" border="0"
      src="img8.png" alt=
      "\begin{displaymath} {\cal L}_{MIX}(\Theta_1,...\Theta_K\vert{\bf y})=\prod_{i=1}^{n}\sum_{k=1}^{K}\pi_kf_k({\bf y_i}\vert\mu_k,\Sigma_k), \end{displaymath}">
 </td>
 <td width="10" align="right">(1)</td></tr></table></div>
 <br clear="all">
  <font size="-1">where 
 <img width="126" height="32" align="middle" border="0" src=
 "img9.png" alt="$\Theta_k=(\mu_k,\Sigma_k,\pi_k)$">
      (<img width="84" height="30" align="middle" border="0" src=
      "img10.png" alt="$k=1,...,K$">) is the set of parameters for
      the <img width="13" height="14" align="bottom" border="0" src=
      "img11.png" alt="$k$">th cluster and <img width="20" height=
      "30" align="middle" border="0" src="img12.png" alt="$f_k$"> is
      the density of the <img width="13" height="14" align="bottom"
      border="0" src="img11.png" alt="$k$">th Gaussion mixture
      component with mean <img width="21" height="29" align="middle"
      border="0" src="img13.png" alt="$\mu_k$"> and the covariance
      <img width="23" height="30" align="middle" border="0" src=
      "img14.png" alt="$\Sigma_k$">. Here, <img width="21" height=
      "29" align="middle" border="0" src="img15.png" alt="$\pi_k$">
      is the probability that an observation belongs to the
      <img width="13" height="14" align="bottom" border="0" src=
      "img11.png" alt="$k$">th cluster with 
 <img width="93" height="40" align="middle" border="0" src=
 "img16.png" alt="$\sum_{k=1}^{K}\pi_k=1$">.</font> 
 <p><font size="-1">Maximization of the likelihood in
 Equation&nbsp;(<a href="#Eq.likelihood">1</a>) provides estimates 
 
 <img width="126" height="38" align="middle" border="0" src=
 "img17.png" alt=
 "$\hat\Theta_k=(\hat\mu_k,\hat\Sigma_k,\hat \pi_k)$"> for the
 unknown parameters of the mixture. The standard choice for fitting
 mixture models is the Expectation-Maximization (EM) algorithm
 [<a href="#DLR77">3</a>] which proved to be
 efficient for our purposes. An important issue in mixture modeling
 is the selection of the number of clusters. Two models
      <img width="30" height="30" align="middle" border="0" src=
      "img18.png" alt="${\cal M}_1$"> and <img width="30" height=
      "30" align="middle" border="0" src="img19.png" alt=
      "${\cal M}_2$"> can be compared by the Bayes factor, that is,
      the posterior odds ratio 
 <img width="143" height="32" align="middle" border="0" src=
 "img20.png" alt=
 "$P({\cal M}_1\vert{\bf y})/P({\cal M}_2\vert{\bf y})$"> for one
 model against the other. We used the Bayesian Information Criterion
 
 <img width="249" height="38" align="middle" border="0" src=
 "img21.png" alt=
 "$ BIC = 2\log[ {\cal L}({\bf y}\vert\hat{\Theta})] - m({\cal M})\log n $">
  to get a computationally efficient approximation of the Bayes
 factor. Here, 
 <img width="54" height="38" align="middle" border="0" src=
 "img22.png" alt="${\cal L}({\bf y}\vert\hat{\Theta})$"> is the
 maximized likelihood, <img width="49" height="32" align="middle"
 border="0" src="img23.png" alt="$m({\cal M})$"> is the number of
 parameters of model <img width="23" height="15" align="bottom"
 border="0" src="img24.png" alt="$\cal M$">.</font></p>
 <p><font size="-1">The fit of a mixture model to an assay puts us
 in the position to quantify the reliability of (i) individual
 observations and (ii) the assay as a whole. Individual observations
 are unreliable, when the uncertainty about assigning the
 observation to a cluster is high. We quantify this by estimating
 the probability of an observation pertaining to one of the
 potential clusters by</font></p>
 <div align="right">
 <table width="100%" align="center">
 <tr valign="middle">
 <td align="center" nowrap><a name="Eq.prob"></a><img width="212"
 height="48" border="0" src="img25.png" alt=
 "\begin{displaymath} p(k\vert{\bf y_i})= \frac{\pi_k f_k({\bf y_i}\vert\mu_k,... ..._{\nu} f_{\nu}({\bf y_i}\vert\mu_{\nu},\Sigma_{\nu})} \end{displaymath}">
 </td>
 <td width="10" align="right">(2)</td></tr></table></div>
 <br clear="all">
  <font size="-1">However, also the assay itself may be unreliable
 and the distinguishability of different clusters may be poor. This
 can be quantified by the overlap 
 <img width="227" height="38" align="middle" border="0" src=
 "img26.png" alt=
 "$ \Psi_{k,l} = \int \sqrt{f_k({\bf y}\vert\Theta_k) f_l({\bf y}\vert\Theta_l)} d{\bf y}\,, $">
  also known as Bhattacharyya distance. The overlap is bounded
 between <img width="12" height="13" align="bottom" border="0"
      src="img27.png" alt="$0$"> and <img width="12" height="13"
      align="bottom" border="0" src="img28.png" alt="$1$">. Two
      identical densities have overlap <img width="12" height="13"
      align="bottom" border="0" src="img28.png" alt="$1$"> while
      infinitely separated clusters have overlap <img width="12"
      height="13" align="bottom" border="0" src="img27.png" alt=
      "$0$">. The smaller the overlap, the more distinguishable are
      the densities. In addition, the overlap allows to compute an
      upper and lower boundary for the probability to decide for the
      wrong cluster (Bayesian probability of error).</font> 
 <h1><a name="SECTION00030000000000000000"><font size=
 "+1">Results.</font></a></h1>
   
 <p><font size="-1">The applicability of GM models to the clustering
 of SNP assay data was tested on a total of 356 non-redundant SNP
 assays generated in the course of a genetic target gene validation
 study. In total, this experimental data set contains 118,016 data
 points and 178 different SNPs. We will discuss the results and
 benefits of using GM models.</font></p>
 <p></p>
 <h2><a name="SECTION00040000000000000000">Bibliography</a></h2>
 <dl compact>
 <dt><a name="BR93">1</a></dt>
 <dd>Banfield, J.&nbsp;D. and Raftery, A.&nbsp;E. 1993. Model based
 Gaussian and non- Gaussian clustering. <em>Biometrics</em>
 49:803-821.</dd>
 <dt><a name="CLK99">2</a></dt>
 <dd>Chen, X., Levine, L. and Kwok, P.&nbsp;Y. 1999. Fluorescence
 polarization in homogeneous nucleic acid analysis. <em>Genome
 Res.</em> 9:492- 498.</dd>
 <dt><a name="DLR77">3</a></dt>
 <dd>Dempster, N., Laird, N. and Rubin, D. 1977. Maximum likelihood
 estimation from incomplete data via the EM algorithm. <em>J. Roy.
 Stat. Soc. B</em> 39:1-38.</dd>
 <dt><a name="GSN02">4</a></dt>
 <dd>Grant, S.&nbsp;F.&nbsp;A., Steinlicht, S., Nentwich, U., Kern,
 R., Burwinkel, B. and Tolle, R. 2002. SNP genotyping on genome wide
 amplified DOP-PCR template. <em>Nucleic Acids Res.</em>
 30:e125.</dd></dl>
  
 <hr>
 <h4>Footnotes</h4>
 <dl>
 <dt><a name="foot97">... Gieger,</a><a href=
 "#tex2html1"><sup>1</sup></a></dt>
 <dd>Fraunhofer Institute for Algorithms and Scientific Computing,
 Schloss Birlinghoven, St. Augustin, Germany. E-mail: 
 <div align="center">
 <b><tt>christian.gieger@scai.fhg.de</tt></b></div></dd>
 <dt><a name="foot98">... Kschischo,</a><a href=
 "#tex2html2"><sup>2</sup></a></dt>
 <dd>University of Applied Sciences Koblenz, RheinAhrCampus,
 Remagen, Germany. E-mail: 
 <div align="center">
 <b><tt>kschischo@rheinahrcampus.de</tt></b></div></dd>
 <dt><a name="foot99">... Kern,</a><a href=
 "#tex2html3"><sup>3</sup></a></dt>
 <dd>LION bioscience AG, Heidelberg, Germany. E-mail: 
 <div align="center">
 <b><tt>rainer.kern@lionbioscience.com</tt></b></div></dd>
 <dt><a name="foot100">... Steinhauser,</a><a href=
 "#tex2html4"><sup>4</sup></a></dt>
 <dd>LION bioscience AG, Heidelberg, Germany. E-mail: 
 <div align="center">
 <b><tt>martin.steinhauser@lionbioscience.com</tt></b></div></dd>
 <dt><a name="foot101">... Tolle</a><a href=
 "#tex2html5"><sup>5</sup></a></dt>
 <dd>PHENEX pharma, Heidelberg, Germany. E-mail: 
 <div align="center">
 <b><tt>ralf.tolle@phenex-pharma.com</tt></b></div></dd></dl>
 <br>
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
