<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>141. A Browser-based Functional Classification SuperViewer for Arabidopsis Genomics</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category05.html'>&lt;&lt;</a> <a href='/recombdb/category06.html'><b>Gene Expression</b> Overview</a> <a href='/recombdb/category07.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-06-140/'>&lt;&lt;</a> Poster 141 [ <a href='poster-06-141.pdf'>PDF</a> | <a href='poster-06-141.doc'>MSWORD</a> ] <a href='/recombdb/poster-06-142/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>141.&nbsp; A Browser-based Functional Classification
 SuperViewer for <i>Arabidopsis</i> Genomics</h2>
 <p align="center"><b>Nicholas Provart<a href="#_ftn1" name=
 "_ftnref1" title="">[1]</a>,&nbsp; Tong Zhu<a href="#_ftn2"
    name="_ftnref2" title="">[2]</a></b></p>
 <p><b>Keywords:</b> functional classification, data visualization,
 Arabidopsis thaliana</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction</b></p>
 <p>Functional classification is commonly applied to data generated
 by microarray gene expression profiling experiments. Indeed, one of
 the most often used bioinformatics tools after BLAST and clustering
 is the functional classification pie chart. However, presentation
 of the absolute numbers of genes in a given cluster falling into
 each functional classification category may be misleading or mask
 difference under a given treatment. Another way of examining this
 sort of information is to normalize to the number of genes in each
 class present on the chip. In this way, differences are more easily
 perceived. We present here a web-based tool, the Functional
 Classification SuperViewer, which performs this normalization,
 bootstraps the dataset to provide a confidence estimate for the
 accuracy of the output, generates a dynamic graph summarizing the
 output for easy incorporation into reports, and provides links to
 TAIR [1] and other databases for individual IDs entered.
 Furthermore, expression values may also be submitted along with the
 IDs, and the values will be displayed with a colour-scale
 background, along with a functional classification bar-code.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Software and files</b></p>
 <p>The program is written in Perl and utilizes GD.pm [2] module for
 dynamic graph generation. Access is via a web-browser. For GeneChip
 IDs or AGI numbers keyed to the Munich Information Center for
 Protein Sequences (MIPS, [3]) dataset, the current MIPS
 classifications for 25450 genes in the MAtDB as of March 13, 2002,
 were downloaded as an&nbsp; XML file&nbsp; from
 http://biors.gsf.de:8111/searchtool/
 searchtool.cgi?request=login_guest&amp;file=frameset. Those IDs
 falling into classification categories other than
 &lsquo;unclassified&rsquo; and &lsquo;classification not yet
 clear-cut&rsquo; were removed from these two categories. Data were
 reformatted into a flat file format which is then used by the
 program during classification. Arabidopsis annotations were
 obtained from The Arabidopsis Information Resource (TAIR, [1]).
 ftp://tairpub:tairpub@ftp.arabidopsis.org/home/tair/ Genes/
 ORF_annotations/ATH1.pep.0172002.total.</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Results and
 Discussion</b></p>
 <p>A class score for normalization was calculated based on the
 following equation. N is number.</p>
 <p>&nbsp;Score <sub>class</sub> =&nbsp; [<i>N</i> <sub>class
 (inputset)</sub>/ <i>N</i> <sub>classified (inputset)</sub>]/[
 <i>N</i> <sub>class (25K)</sub>/ <i>N</i> <sub>classified
 (25K)</sub>]</p>
 <p>Furthermore, the input set was bootstrapped one hundred times by
 sampling the input set (with repeats) and then reclassifying each
 set so generated. The standard deviation for the scores generated
 from the bootstrap sets is displayed along with the normalized
 class score. In this way classes represented by small numbers of
 genes on the chip may be easily identified (these tend to generate
 high scores or low scores if over- or under-represented but the SD
 will show them to be spurious).</p>
 <p>Below is an example of the dynamically generated output. Left is
 shown the non-normalized numbers, right an example of normalized
 scores. As may be seen, it is readily apparent that protein
 biosynthesis is being actively up-regulated in this example. The
 gene list for this example was generated from a microarray
 experiment looking at chilled wild-type Arabidopsis plants, and
 such a response is well-known in other organisms.</p>
 <p align="center"><img width="457" height="171" src=
 "image002.jpg"></p>
 <p align="center"><br>
  Figure 1: Example Output of the Functional Classification
 SuperViewer showing non-normalized (left panel) and normalized
 (right panel) results.</p>
 <p>The HTML output also contains an overview table which indicates
 which classes a given AGI numbered protein falls into. Furthermore,
 links are also provided to TAIR and GenBank IDs, and it is also
 possible to upload expression values associated with each AGI
 number in the list. The resultant table may be easily copied into
 Word or other applications.</p>
 <p><img width="496" height="150" src="image004.jpg"></p>
 <p align="center">Table 1:&nbsp; Example HTML output from
 SuperViewer.</p>
 <p><b>4&nbsp; References</b></p>
 <p>[1] The Arabidopsis Information Resource.
 http://www.arabidopsis.org/</p>
 <p>[2] http://stein.cshl.org/WWW/software/GD/</p>
 <p>[3] Mewes, H.W., Heumann, K., Kaps, A., Mayer, K., Pfeiffer, F.,
 Stocker, S. and Frishman, D. 1999. MIPS: a database for genomes and
 protein sequences. <i>Nucleid Acids Research</i> 27:
 44-48.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a> Department of
 Botany, University of Toronto, 25 Willcocks St., Toronto, ON. M5S
 3B2, CANADA. E&#8209;mail: provart@botany.utoronto.ca</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Torrey Mesa
 Research Institute of Syngenta, AG. 3115 Merryfield Row, San Diego,
 CA. 92121, USA. E-mail: tong.zhu@syngenta.com</p></div></div>
 </body>
 </html>
