<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>85. Functional Fingerprints of Folds: Evidence for Correlated Structure-Function Evolution.</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category03.html'>&lt;&lt;</a> <a href='/recombdb/category04.html'><b>Molecular Evolution</b> Overview</a> <a href='/recombdb/category05.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-04-083/'>&lt;&lt;</a> Poster 085 [ <a href='poster-04-085.pdf'>PDF</a> | <a href='poster-04-085.doc'>MSWORD</a> ] <a href='/recombdb/poster-04-086/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>85. Functional Fingerprints of Folds: Evidence for Correlated
 Structure-Function Evolution.</h2>
 <p align="center"><b>Boris E. Shakhnovich<a href="#_ftn1" name=
 "_ftnref1" title="">[1]</a>, Nikolay V. Dokholyan<a href="#_ftn2"
 name="_ftnref2" title="">[2]</a>, Charles</b></p>
 <h3>DeLisi<a href="#_ftn3" name="_ftnref3" title="">[3]</a> and
 Eugene I. Shakhnovich<a href="#_ftn4" name="_ftnref4" title=
 "">[4]</a></h3>
 <p><b>Keywords:</b> molecular evolution, structure-function, power
 laws.</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction</b></p>
 <p>Using structural similarity clustering of protein domains:
 Protein Domain Universe Graph (PDUG), and a hierarchical functional
 annotation: Gene Ontology (GO) as two evolutionary lenses, we find
 that each structural cluster (domain fold) exhibits a distribution
 of functions that is unique to it.&nbsp; These functional
 distributions are functional fingerprints that are specific to
 characteristic structural clusters and vary from cluster to
 cluster. Furthermore, as structural similarity threshold for domain
 clustering in PDUG is relaxed we observe an influx of
 earlier-diverged domains into clusters. These domains join clusters
 without destroying the functional fingerprint. These results can be
 understood in light of a divergent evolution scenario that posits
 correlated divergence of structural and functional traits in
 protein domains from one or few progenitors.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Main Results</b></p>
 <p>The main result of this work is a striking finding that the
 corollary relationship between structural evolution and acquisition
 of new function by protein domains necessitated by a divergent
 evolution scenario can be <i>quantitatively</i> observed on the
 PDUG. Looking at PDUG through a hierarchical description of
 structural comparisons we find that we can characterize different
 clusters by the ``functional fingerprint'' that they display. A
 functional fingerprint is the distribution of functions within a
 particular cluster. We find that this distribution is quite unique
 to a given fold family on levels of functional annotation with high
 specificity of description. If we relax the Z<sub>min</sub>
 threshold, we can see an influx of protein domains into
 structurally similar clusters. Even with the influx of as many as
 one hundred percent of the earlier diverged domains into the
 ancestral cluster, the functional fingerprint is not destroyed
 (does not become random), but is complemented with the same and
 similar functions. This preservation of unique functional
 fingerprints through evolutionary dynamics further highlights the
 close relationship between structure and function necessitated by
 divergent evolution.</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Figures and tables</b></p>
 <p>Fig 1 (below) Functional fingerprints of folds. X-axis is the
 functional annotation category. Y-axis is the normalized number of
 proteins annotated with that function. (a-c) Functional annotation
 at the fifth level of GO ontology for Z<sub>min</sub> = 9. Notably,
 each cluster has its own, distinct functional fingerprint that is
 observably different than those of other clusters. (d-f) At the
 fifth level of annotation after proteins joined their ancestral
 clusters, for Z<sub>min</sub>= 2. The fingerprints are more
 diverse, however still differ significantly from each other. (g-i)
 At the first level of annotation with Z<sub>min</sub> = 2, the
 fingerprints overlap significantly, and hard to distinguish one
 from the other</p>
 <p><img width="144" height="197" src=
 "image002.gif">&nbsp;<img width="151" height="200" src=
 "image004.gif">.<img width="155" height="199" src=
 "image006.gif"></p>
 <p>Fig 2</p>
 <p><img width="214" height="156" src="image008.jpg" align="left"
 hspace="12">A schematic picture of how protein domains diffuse into
 structural clusters and define them at different thresholds. Nodes
 represent protein domains, links represent structural similarity at
 above threshold. Thick line represents domains connected at
 Z<sub>min</sub> = 15, dotted lines represent domains connected at
 Z<sub>min</sub> = 11 and thin lines represent domains connected at
 Z<sub>min</sub>=9. Domains inside PDUG have drastically different
 connectivity depending on the similarity threshold based on which
 we connect two nodes. It is evident that at a very high threshold
 Z<sub>min</sub>=inf, all protein domains are orphans, i.e. they are
 connected to no other domains because their similarity is lower
 than the structural similarity threshold. At Z<sub>min</sub>= 0 all
 protein domains are connected because the threshold is
 non-discriminatory. As we change the threshold, we observe orphans
 diffuse into larger clusters of sufficiently similar domains.&nbsp;
 As the evolutionary threshold Z<sub>min</sub> is relaxed, proteins
 that were orphans and had no structural neighbors join their
 ancestral clusters. All proteins in PDUG have less that 25\%
 sequence homology to each other.&nbsp; In this figure, the domains
 that joined at Z<sub>min</sub>=15 are yellow, at Z<sub>min</sub>=11
 are in beige and at Z<sub>min</sub>=9 are white.</p>
 <p><b>4&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>References and
 bibliography</b></p>
 <p>Chothia, C. and A. M. Lesk (1986). "The relation between the
 divergence of sequence and structure in proteins." <u>Embo J</u>
 <b>5</b>(4): 823-6.</p>
 <p>Dietmann, S., J. Park, et al. (2001). "A fully automatic
 evolutionary classification of protein folds: Dali Domain
 Dictionary version 3." <u>Nucleic Acids Res</u> <b>29</b>(1):
 55-7.</p>
 <p>Dokholyan, N. V., B. Shakhnovich, et al. (2002). "Expanding
 protein universe and its origin from the biological Big Bang."
 <u>Proc Natl Acad Sci U S A</u> <b>99</b>(22): 14132-6.</p>
 <p>Holm, L. and C. Sander (1996). "Mapping the protein universe."
 <u>Science</u> <b>273</b>(5275): 595-603.</p>
 <p>Murzin, A. G. (1998). "How far divergent evolution goes in
 proteins." <u>Curr Opin Struct Biol</u> <b>8</b>(3): 380-387.</p>
 <p>Teichmann, S. A., A. G. Murzin, et al. (2001). "Determination of
 protein function, evolution and interactions by structural
 genomics." <u>Curr Opin Struct Biol</u> <b>11</b>(3):
 354-63.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a>&nbsp;
 Bioinformatics Department, Boston University, Boston, MA 02215,
 USA, E-mail: borya@acs.bu.edu</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a>&nbsp;
 Department of Biochemistry and Biophysics, University of North
 Caroline at Chapel Hill School of Medicine, Chapel Hill, NC 27599,
 USA E&#8209;mail: dokh@med.unc.edu</p></div>
 <div id="ftn3">
 <p><a href="#_ftnref3" name="_ftn3" title="">[3]</a>&nbsp;
 Bioinformatics Department, Boston University, Boston, MA 02215,
 USA, E-mail: delisi@acs.bu.edu</p></div>
 <div id="ftn4">
 <p><a href="#_ftnref4" name="_ftn4" title="">[4]</a> Department of
 Chemistry and Chemical Biology, Harvard University,&nbsp;
 Cambridge, Ma 02138, USA E-mail:
 Eugene@belok.harvard.edu</p></div></div>
 </body>
 </html>
