<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 
 <html>
 <head>
 
 <title>96. Accelerating Screening of 3D Protein Data with a Graph Theoretical Approach</title>
 
 
 
 
 
 
 
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category04.html'>&lt;&lt;</a> <a href='/recombdb/category05.html'><b>Protein Structure</b> Overview</a> <a href='/recombdb/category06.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-05-095/'>&lt;&lt;</a> Poster 096 [ <a href='poster-05-096.pdf'>PDF</a> | <a href='poster-05-096.tgz'>TeX</a> | <a href='poster-05-096.ps'>PostScript</a> ] <a href='/recombdb/poster-05-098/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div align="center"><font size="+2"><b>96. Accelerating Screening
 of 3D Protein Data with a Graph Theoretical
 Approach<br></b></font></div>
 <div align="center"><font size="+2"><b><br>
 <br></b></font></div>
 <div align="center"><b>Cornelius Fr&ouml;mmel<a name="footfnm1"
 href="#fnm1"><sup>1</sup></a>, Andrean Goede<a name="tex2html1"
 href="#footfnm1"><sup>1</sup></a> , Clemens Gr&ouml;pl<a name=
 "footfnm2" href="#fnm2"><sup>2</sup></a>,
 Stefan&nbsp;Hougardy<a name="tex2html2" href=
 "#footfnm2"><sup>2</sup></a> , Till Nierhoff<a name="tex2html3"
 href="#footfnm2"><sup>2</sup></a> <img width="9" height="15"
      align="bottom" border="0" src="img1.png" alt="$ ^,$"><a name=
      "footfnm3" href="#fnm3"><sup>3</sup></a>, Robert
      Preissner<a name="tex2html4" href="#footfnm1"><sup>1</sup></a>
      , Martin Thimm<a name="tex2html5" href=
      "#footfnm2"><sup>2</sup></a></b></div>
 <p><font size="-1"><br>
 <br></font></p>
 <p><font size="-1"><b>Keywords:</b> superposition, data analysis,
 data mining, DIP</font></p>
 <h1><a name="SECTION00010000000000000000"><font size=
 "+1">Introduction</font></a></h1>
 <p><font size="-1">The Dictionary of Interfaces in Proteins (DIP)
 is a data base containing information about interacting parts of
 the three-dimensional structure in proteins that are called patches
 [1]. A patch is a set of atoms of a secondary structural element
 (SSE) that are in spatial proximity with atoms of another structure
 (SSE, solvent, or ligand). The spatial similarity between two
 patches can be expressed in terms of a 3D similarity score. Given
 two patches, this score is computed by choosing subsets of the
 atoms in both and then finding a rigid motion that minimizes the
 root mean square distance (rms) of the chosen atoms. The score
 value is then</font> 
 </p>
 <div align="center"><img width="53" height="32" align="middle"
 border="0" src="img2.png" alt=
 "$\displaystyle s \cdot \exp($">rms/&Aring;<img width="15"
      height="32" align="middle" border="0" src="img3.png" alt=
      "$\displaystyle ), $"></div>
  <font size="-1">where <img width="11" height="13" align="bottom"
 border="0" src="img4.png" alt="$ s$"> is the number of chosen atoms
 divided by the size of the smaller patch. This score acts as an
 assessment criterion for the prediction of binding sites or protein
 folding. Currently, only a heuristic value is computed, because the
 problem is hard.</font> 
 <p><font size="-1">DIP is a growing database that currently
 contains more than <img width="27" height="16" align="bottom"
 border="0" src="img5.png" alt="$ 10^6$"> patches of several
 thousands of proteins. Its purpose is to serve as a repository in
 which patches similar to given query patches can be found. However,
 even the computation of a heuristic value for one similarity score
 takes about one tenth of a second. As a consequence, a comparison
 of a query patch to all patches in DIP already at present takes
 more than one day. With the growth of DIP more efficient search
 strategies are therefore necessary. In this work we address two
 questions:</font></p>
 <ol>
 <li>Given a query patch, how similar are other patches in terms of
 the score that are relevant to the query.</li>
 <li>Given a query patch, is there a way to search for the relevant
 patches, scanning only a small part of the database.</li></ol>
 <h1><a name="SECTION00020000000000000000"><font size=
 "+1">Results</font></a></h1>
 <p><font size="-1">As to the first question, it turns out that
 there are essentially three ranges of the similarity score which
 can be attributed with (i)&nbsp;noise, (ii)&nbsp;accidental
 similarity, and (iii)&nbsp;homology, respectively. We consider the
 similarity graph, i.e. the graph whose vertices correspond to
 patches and which has an edge between two patches if and only if
 their similarity score exceeds a certain threshold value. We find
 that the size of the largest connected component of the similarity
 graph, plotted against the threshold value, has two remarkable
 bends at the values which separate the three ranges described
 above. In contrast to this, the same plot has no bends at all if
 the score values are randomly permuted among the patch pairs
 (Figure&nbsp;1). We infer that the similarity data has structural
 properties that might be used to address the second
 question.</font></p>
 <p><font size="-1">We have tested several approaches to reduce the
 number of necessary comparisons for a given query patch. They have
 been measured in terms of a speed up factor that is defined as
 follows: If the number of patches in the data set is
      <img width="14" height="13" align="bottom" border="0" src=
      "img6.png" alt="$ n$">, then the speed up factor is
      <img width="14" height="13" align="bottom" border="0" src=
      "img6.png" alt="$ n$"> divided by the number of comparisons
      needed by the procedure to identify those entries that have a
      similarity score with the query in one of the ranges (ii) or
      (iii). So, the speed up factor measures how many times faster
      than linear search the tested procedures are.</font></p>
 <p><font size="-1">For the range (iii), corresponding to homologous
 patch pairs, a hierarchical organization of the data leads to a
 speed up factor of the order of magnitude of <img width="57"
      height="32" align="middle" border="0" src="img7.png" alt=
      "$ n/\log n$">. Range (ii) is especially interesting, because
      the patch pairs in this range that are not homologous
      nevertheless share some structural features. These
      exceptionally similar pairs can provide interesting
      information e.g. about ancestral relationships and can only be
      found using folding information about the proteins. We found
      that this range is more difficult to handle and that
      substantial acceleration can only be achieved at the expense
      of false negatives. We have developed a search structure that
      is efficiently computable and that could be tuned to find 85%
      of the relevant patches with a speed up of ten or 95% with a
      speed up of four (Figure&nbsp;2).</font></p>
 <table width="317">
 <tr>
 <td><br>
 <br>
 <br>
 <br>
 <br>
 <br>
  <img width="226" height="217" align="bottom" border="0" src=
 "img9.png" alt="\includegraphics{poster-05-096-fig03.eps}"> 
 <p><br>
 <img width="253" height="70" align="middle" border="0" src=
 "img8.png" alt=
 "$\textstyle \parbox{5.5cm}{\footnotesize Figure 1: Size of the largest component... ... value for patches of at least 100 atoms. Actual and randomly permuted data.}$">
 </p></td></tr></table>
  
 <table width="317">
 <tr>
 <td><br>
  <img width="227" height="159" align="bottom" border="0" src=
 "img11.png" alt="\includegraphics{poster-05-096-fig02.eps}">
 <img width="227" height="159" align="bottom" border="0" src=
 "img12.png" alt="\includegraphics{poster-05-096-fig01.eps}"> 
 <p><br>
 <img width="253" height="58" align="middle" border="0" src=
 "img10.png" alt=
 "$\textstyle \parbox{5.5cm}{\footnotesize Figure 2: Performance of a search struc... ...range (ii). Percentage found and speed up factor for two parameter settings.}$">
 </p></td></tr></table>
 <h1><a name="SECTION00030000000000000000"><font size=
 "+1">References</font></a></h1>
 <p><font size="-1">(1) R. Preissner, A. Goede, and C. Froemmel.
 Dictionary of Interfaces in Proteins (DIP). Data Bank of
 Complementary Molecular Surface Patches. <i>J. Mol. Biol.</i>,
 280:535-550, 1998.</font></p>
 <hr>
 <h4>Footnotes</h4>
 <dl>
 <dt><a name="fnm1">...&ouml;mmel</a><a href=
 "#footfnm1"><sup>1</sup></a></dt>
 <dd>Institut f&uuml;r Biochemie, Charit&eacute;, Monbijoustr. 2,
 10117 Berlin, {cornelius.froemmel, andrean.goede,
 robert.preissner}@charite.de. Supported by the Berlin Center for
 Genome Based Bioinformatics (BCB).</dd>
 <dt><a name="fnm2">...&ouml;pl</a><a href=
 "#footfnm2"><sup>2</sup></a></dt>
 <dd>Institut f&uuml;r Informatik, Humboldt-Universit&auml;t zu
 Berlin, 10099 Berlin, {groepl, hougardy, nierhoff,
 thimm}@informatik.hu-berlin.de. Clemens Gr&ouml;pl is supported by
 Deutsche Forschungsgemeinschaft (DFG), grant no. Pr 296/6-3. Martin
 Thimm is supported by the DFG Research Center "Mathematics for key
 technologies" (FZT&nbsp;86) in Berlin.</dd>
 <dt><a name="fnm3">...<img width="9" height="15" align="bottom"
 border="0" src="img1.png" alt="$ ^,$"></a><a href=
 "#footfnm3"><sup>3</sup></a></dt>
 <dd>corresponding author</dd></dl>
 <br>
 <hr>
 <address>2003-04-07</address>
 </body>
 </html>
