<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>83. Stochastic Mechanisms of Evolution: The Number of Domains-to-Protein Links Predicts the Number of Genes in Genomes</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center"> <a href='/recombdb/category03.html'>&lt;&lt;</a> <a href='/recombdb/category04.html'><b>Molecular Evolution</b> Overview</a> <a href='/recombdb/category05.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-04-082/'>&lt;&lt;</a> Poster 083 [ <a href='poster-04-083.pdf'>PDF</a> | <a href='poster-04-083.doc'>MSWORD</a> ] <a href='/recombdb/poster-04-085/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>83. Stochastic Mechanisms of Evolution:<br>
  The Number of Domains-to-Protein Links Predicts the Number of
 Genes in Genomes</h2>
 <p align="center"><b>Vladimir Kuznetsov<a href="#_ftn1" name=
 "_ftnref1" title="">[1]</a>, Valery Pickalov<a href="#_ftn2"
    name="_ftnref2" title="">[2]</a></b></p>
 <p><b>Keywords:</b> proteome complexity, evolution, number of
 genes, stochastic processes, Kolmogorov-Waring distribution</p>
 <p><b>1&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Introduction</b></p>
 <p>Recently, we proposed a novel statistically-justifiable approach
 for estimating the number of&nbsp; protein-coding genes in
 partially-sequenced and partially-annotated genomes&nbsp; [1-4].
 Using different data sets, we have estimated that there are
 32,200-42,700 genes in the human genome [3-5]. Our gene number
 prediction methodology employs information about the number of
 evolutionary conserved protein-coding sequences (protein
 domains/families/motifs) distributed in proteins within a sample
 proteome from large number of fully sequenced genomes.&nbsp;</p>
 <p>In this particular work, we developed the evolution analysis
 that consists of revisiting the protein-coding gene count for
 human, mouse and <i>Arabidopsis thaliana</i>.</p>
 <p><b>2&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Methods and Software</b></p>
 <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; We have analyzed the
 statistical distributions of the number of the non-redundant
 domain-to-protein links of the InterPro domains (protein domains,
 families, motifs [<a href=
 "http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>
 ; updated on Dec 16, 2002]) in proteins of the sample proteomes of
 the 90 fully sequenced genome organisms of three kingdoms of life:
 Archaea, Bacteria and Eukaryota [<a href=
 "http://www.ebi.ac.uk/proteome/ProteomeSource.html">http://www.ebi.ac.uk/proteome/ProteomeSource.html</a>/;
 updated on Dec 17, 2002]</p>
 <p><b>3&nbsp;&nbsp;&nbsp;&nbsp;</b> <b>Results and
 Discussion</b></p>
 <p>We found that many protein domains that appear in nature are
 widely distributed in phylogenetically distinct species and
 progressively appear in new proteins due to evolution. A large
 fraction of the identified distinct InterPro domains (i.e. unique
 domains, homologous domain families, motifs) in these 90 proteomes
 (1409 (23%) out of 6085 domains) are found in at least one organism
 in each of these kingdoms of life. The 1666 (49%) out of 3427 human
 protein domains are found in at least one of the archaeal or
 bacterial organism. We found similar numbers of domains shared with
 Archaea and Bacteria for mouse and the <i>A. thaliana</i> domain
 sets. We counted the number of non-redundant occurrences of the
 given distinct domain in all proteins of the organisms. By
 non-redundant, we mean that a distinct domain is counted only once
 even if it occurs several times in the same multi-domain protein.
 We referred to this number as the <i>number of&nbsp; non-redundant
 domain-to-protein links</i> of a distinct domain in the proteome of
 the organism.</p>
 <p>All the observed domain-to-protein link frequency distributions
 for these 90 proteomes are universally well fitted by a family of
 skewed Pareto-like probability functions (the Waring, Simon-Yule,
 Champernowne Zipf distributions). These probability functions are
 relatively robust to experimental errors. The family of these
 functions is associated with the probabilistic model of progressive
 domains evolution which is a steady state distribution of a linear
 birth-death Markov random process described by the forward
 Kolmogorov equations [5]. Note, this evolution model does not
 explicitly include the selection processes. However, the birth and
 death parameters of the model could be driven by the selection
 forces. Our statistical analysis of the distributions associated
 with steady-state solution of the model implies certain simple
 <i>scale-invariant</i> rules of progressive evolution for the
 number of protein domains and their occurrences in proteins within
 the proteomes. In particular, we found strong linear relationship
 between the number of protein-coding genes, G, and the total number
 of non-redundant domain-to-protein links in a sample proteome, M
 (r=0.998). This relationship was fitted by the regression model
 G=0.8506M for the 87 proteomes (excluding mouse, <i>A. thaliana</i>
 and human) and through the use of the extrapolations this model
 predicts 33560, 25910, and 26870 protein-coding genes in human,
 mouse and <i>A. thaliana</i> proteome, respectively. The standard
 errors for these estimates are 60, 50 and 50 domains,
 respectively.&nbsp; We also found strong linear relationship
 between the number of protein-coding genes, G, the number of
 non-redundant domain-to-protein links for mostly redundant domain
 in a proteome,&nbsp; the total number of non-redundant
 domain-to-protein links in subsets of the 100 or 300 most widely
 occurring protein domains found in&nbsp; studied fully-sequenced
 genomes. Correlation coefficients are 0.93, 0.993 and 0.996,
 respectively. In the latter two cases, the 33900 and 34720 human
 genes are predicted by the linear regression models,
 respectively.&nbsp; Through counting of the mean value by our three
 estimates, we predicted that there are 34060 protein-coding genes
 in the human genome.</p>
 <p><b>4&nbsp;&nbsp;&nbsp; Conclusions</b></p>
 <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Proteomes of
 the diverse organisms can be specifically related to one another in
 terms of the distributions of the&nbsp; number of non-redundant
 domain-protein links.</p>
 <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Our
 whole-genome comparison of domain occurrences in Archaea, Bacteria
 and Eukaryota proteomes implies even if the size of the genome is
 large, the number of protein-coding genes in the organism is
 determined by a small number (300 or less) of the evolutionary
 conserved and functionally essential sequences (domains) contained
 within proteins of many studied organism.</p>
 <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Our statistical
 modeling and observations suggest a&nbsp; random component in
 innovation, birth and death processes for vast majority of
 evolutionary conserved protein-coding sequences as well as positive
 selection and&nbsp; preferential propagation of certain fraction
 of&nbsp; already existing conserved protein domains in certain
 species.</p>
 <h1>5&nbsp;&nbsp;&nbsp; References</h1>
 <p>[1] Kuznetsov, V.A. 2001. Distribution associated with
 stochastic processes of gene expression in a single eukaryotic
 cell. <i>EURASIP J. on Applied Signal Processing</i>&nbsp;
 <b>4</b>&nbsp; pp. 285-296.</p>
 <p>[2] Kuznetsov, V.A., Knott, G.D., Bonner, R.F. 2002. General
 statistics of stochastic process of gene expression in eukaryotic
 cells. <i>Genetics</i> <b>161</b>, 1321-1332.</p>
 <p>[3] Kuznetsov, V.A. 2002.&nbsp; How can we quantitatively
 describe growth of proteome complexity due to evolution? In:
 Florea, L., Walenz, B., Hannenhalli, S., editors, <i>Currents in
 Computational Molecular Biology</i> <i>(RECOMB 02)</i>, Washington,
 pp. 113-114.</p>
 <p>[4] Kuznetsov, V.A. 2002. Statistics of the numbers of
 transcripts and protein sequences encoded in the genome, In: Zhang,
 W., Shmulevich, I., editors<i>, Computational and Statistical
 Methods to Genomics</i>,&nbsp; Boston: Kluwer, pp. 125-171.</p>
 <p>[5] Kuznetsov, V.A, Pickalov, V.V., Senko, O.V., and Knott, G.D.
 2002. Analysis of the evolving proteomes: Predictions of the number
 of protein domains in nature and the number of genes in eukaryotic
 organisms. <i>J. Biol.&nbsp; Systems</i> <b>10</b> &nbsp;pp.
 381-408.</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a> Laboratory
 Integrative and Medical Biophysics, NICHD/NIH, Bethesda, MD, 20892.
 E-mail: <a href="mailto:vk28u@nih.gov">vk28u@nih.gov</a></p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Institute of
 Theoretical and&nbsp; Applied Mechanics SB RAS, Novosibirsk,
 630090, Russia. E-mail:&nbsp; <a href=
 "mailto:pickalov@itam.nsc.ru">pickalov@itam.nsc.ru</a></p></div></div>
 </body>
 </html>
