<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
 
 <title>16. Amino acid composition of genomes, lifestyles of organisms and evolutionary trends: a large-scale analysis</title>
 </head>
 <body bgcolor="#ffffff" alink="#993333" link="#993333" vlink="#993333">
<table bgcolor="#000000" width="100%" cellspacing=0 cellpadding=0 border=0><tr><td><table width="100%" cellspacing=2 cellpadding=4 border=0>
<tr bgcolor="#bbbbbb"><td align="center"> <a href='/recombdb/'>Recomb 2003 PosterDB Overview</a> </td></tr>
<tr bgcolor="#cccccc"><td align="center">  <a href='/recombdb/category01.html'><b>Genomics</b> Overview</a> <a href='/recombdb/category02.html'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#dddddd"><td align="center"> <a href='/recombdb/poster-01-014/'>&lt;&lt;</a> Poster 016 [ <a href='poster-01-016.pdf'>PDF</a> | <a href='poster-01-016.doc'>MSWORD</a> ] <a href='/recombdb/poster-01-017/'>&gt;&gt;</a> </td></tr>
<tr bgcolor="#eeeeee"><td align="center"> 
<form method="post" action="/cgi-bin/htsearch">
<font size="-1">
Search Recomb2003 PosterDB:
<input type="text" size="30" name="words" value="">
<input type="submit" value="Search">
&nbsp;
Match: <select name="method">
<option value="and">All
<option value="or">Any
<option value="boolean">Boolean
</select>
</form>
</td></tr>
</table>
</td></tr>
</table>
<br>





 <div>
 <h2>16.&nbsp; Amino acid composition of genomes, lifestyles of
 organisms and evolutionary trends: a large-scale analysis</h2>
 <p align="center"><b>Fredj Tekaia<a href="#_ftn1" name="_ftnref1"
 title="">[1]</a>, Edouard Yeramian<a href="#_ftn2" name="_ftnref2"
 title="">[2]</a> and Bernard Dujon<sup>1</sup></b></p>
 <p><b>&nbsp;</b></p>
 <p><b>Key words&nbsp;</b>: hyperthermophiles, mesophiles,
 thermostability,&nbsp; correspondence analyses.</p>
 <p><b>1 Introduction</b></p>
 <p>Global understanding of lifestyle properties characterizing
 living organisms through relevant genomic signatures is a major aim
 of genome projects [1,2]. Considering amino acid composition in 55
 predicted proteomes (See Figure 1) a large-scale analysis has been
 performed using correspondence analysis [4]. The obtained
 distribution allows to gain better understanding of the relative
 importance for the various components involved in the intricate
 relations connecting amino acid compositions with other
 biologically important features including lifestyle and
 evolutionary trends.</p>
 <p>&nbsp;</p>
 <p><b>2 Materials and methods</b></p>
 <p>A matrix T of 55 species and 20 amino-acids was constructed
 (with T<sub>ij</sub> defined as the number of occurences of the
 amino-acid j in the species i, divided by the total number of amino
 acids in i and multiplied by 100). Three dummy variables and eleven
 observations were added by considering corresponding average
 amino-acid compositions. The dummy variables are:
 &lsquo;char&rsquo; for charged amino acids, &lsquo;pol&rsquo; for
 polar/uncharged amino acids&nbsp; and &lsquo;hyd&rsquo; for
 hydrophobic amino-acids. The dummy observations are: MES, TH, HTH
 denoting respectively mesophilic, thermophilic and the
 hyperthermophilic species; and&nbsp; SPC, E, A, B, EA, EB, AB and
 EAB denoting respectively genes specific to each species, genes
 conserved exclusively in one domain of life [E&nbsp;: in eukaryal
 species, A&nbsp;: in archaeal species and B&nbsp;: in bacterial
 species]; genes conserved in a combinaison of 2 domains [EA&nbsp;:
 eukaryal and archaeal,&nbsp; EB&nbsp;: eukaryal and bacterial and
 AB&nbsp;: archaeal and bacterial] and in the intersection of the
 three domains [EAB&nbsp;: eukaryal, archaeal and bacterial].&nbsp;
 These sets of genes are obtained by large-scale proteome
 comparisons as described in [5,6]. Correspondence analysis is then
 performed on the obtained 66 lines and&nbsp; 23 columns matrix (see
 <a href=
 "http://www-alt.pasteur.fr/~tekaia/genomesaa.html">http://wwwalt.pasteur.fr/~tekaia/genomesaa.html</a><a name="_Hlt27975715">
 </a><a name="_Hlt531517933"></a>)<b>.</b></p>
 <p><b>3 Results</b> (see Figure 1)</p>
 <p>1)Correspondence analysis of the amino acid compositions allows
 to discriminate between the three known lifestyles
 (mesophily/thermophily/hyperthermophily)&nbsp;; 2) For various
 organisms, amino-acid composition properties are essentially driven
 by GC content, and to a significantly lesser extent by growth
 temperatures associated with lifestyles. The respective
 contributions of these two components are 61% and 17%. These
 proportions are essentially unchanged with respect to a previous
 analysis [7], which involved only 15 genomes, available at the
 time; 3) In terms of amino acid compositional biases, two specific
 &lsquo;signatures&rsquo; for thermophily (in a broad sense,
 including hyperthermophily) can be detected. First, <i>thermophilic
 species display a relative abundance in glutamic acid,
 concomitantly with the depletion in glutamine</i>. Second, in
 thermophilic species, the relative abundance in glutamic acid
 (negative charge) is significantly correlated (r=0.83 with
 p&lt;0.0001), with the increase in the lumped &lsquo;pool&rsquo;
 Lysine+Arginine (positive charges). This correlation (absent in
 mesophiles) could be interpreted on physico-chemical bases,
 relevant to the thermostability of proteins;&nbsp; 4) A
 statistically significant difference is observed between the
 average lengths of the genes in the surveyed species, following
 their distribution between the three domains of life. Also a
 significant difference is observed between the average lengths of
 themophilic (283.0 <u>+</u> 5.8) versus mesophilic (340 <u>+</u>
 9.4) genes. It is thus possible that the general shortening of the
 primary sequences in thermophilic proteins plays a role in
 thermostability&nbsp;; 5) Considering various combinations of
 conservation properties (genes conserved exclusively in Eukaryotes,
 in Archaea, in Bacteria, in combinations of two and three domains)
 correspondence analysis reveals a trend towards
 thermophilic-hyperthermophilic profiles for the most conserved
 subset of genes (ancient genes)&nbsp;; 6) When limited to the
 subset of &ldquo;species-specific&rdquo; genes, correspondence
 analysis leads to a different picture for the clustering of genomes
 following amino-acid compositions&nbsp;(not shown): for
 example,&nbsp; the &lsquo;core&rsquo; specific part of a genome can
 bear lifestyle signatures different from those of the complete
 genome.</p>
 <p>7) Detailed material and results are in [8] and in in :
    <a href="http://www-alt.pasteur.fr/~tekaia/sacso.html">http://www-alt.pasteur.fr/~tekaia/sacso.html</a></p>
 <p><b>4 References&nbsp;:</b></p>
 <p>[1] Rothschild, L.J., Mancinelli, R.L., 2001. Life in extreme
 environments. <i>Nature</i> 409, 1092-1101.</p>
 <p>[2] Haney, P.J., Badger, J.H., Buldak, G.L., Reich, C.I., Woese,
 C.R., Olsen, G.J., 1999. Thermal adaptation analyzed by comparison
 of protein sequences from mesophilic and extremely thermophilic
 <i>Methanococcus</i> species. <i>Proc. Natl. Acad. Sci.</i> USA 96,
 3578-3583.</p>
 <p>[3] Natale, D.A., Shankavaram, U.T., Galperin, M.Y., Wolf, Y.I.,
 Aravind, L., Koonin, E.V., 2000. Towards understanding the first
 genome sequence of a crenarchaeon by genome annotation using
 clusters of orthologous groups of proteins (COGs). <i>Genome
 Biology</i> 5,
 1-19.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
 <p>[4] Benzecri, J-P., 1973. L'analyse des donn&eacute;es. Vol 2:
 L'analyse des correspondances. <i>Dunod</i>, Paris, France.</p>
 <p>[5] Tekaia, F., Dujon, B., 1999. Pervasiveness of gene
 conservation and persistence of duplicates in cellular genomes.
 <i>J. Mol. Evol.</i> 49, 591-600.</p>
 <p>[6] Tekaia, F., Lazcano, A., Dujon, B., 1999. The genomic tree
 as revealed from whole proteome comparisons. <i>Genome Research</i>
 9, 550-557.</p>
 <p>[7] Cole, S.T., et al., 1998. Deciphering the biology of
 <i>Mycobacterium tuberculosis</i> from the complete genome
 sequence. <i>Nature</i> 393, 537-544.</p>
 <p><img width="326" height="188" src="image002.gif" align="left"
 hspace="12" alt="Text Box: ">[8] Tekaia, F., Yeramian, E. and Dujon
 B. 2002. Amino acid composition of genomes, lifestyle of organisms
 and evolutionary trends&nbsp;: a global picture with correspondence
 analysis. <i>Gene</i>. 297, 51-60.</p>
 <p><b>5 Figure 1.</b> Distribution of species according to
 amino-acid compositions, as obtained on the first factorial space
 (F<sub>1</sub> and F<sub>2</sub>) by cor&shy;respondence&nbsp;
 analysis. This plane represents 77,5% of the total information
 embedded in the original data matrix. The relative amino acid
 compositions were calculated from the published predicted proteomes
 of the following species (each genome is indicated on the graph by
 an abbreviation).</p>
 <p>Eukaryota (7 genomes)&nbsp;: <i>Arabidopsis thaliana</i> (ath),
 <i>Candida albicans</i> (cacd), <i>Caenorhabditis elegans</i> (ce),
 <i>Drosophila melanogaster</i> (dm), <i>homo sapiens</i> (hs),
 <i>Saccharomyces cerevisiae</i> (sc), <i>Schi&shy;zosaccharomyces
 pombe</i> (sp).</p>
 <p>Archaea (14 genomes)&nbsp;: <i>Aeropyrum pernix</i> K1 (ape), (a
 modified version [3] is also used and is denoted apem),
 <i>Archaeoglobus fulgidus</i> (af), <i>Halobacterium</i> sp. NRC-1
 (h), <i>Methanobacterium thermoautotrophicum</i> (mth),
 <i>Methanococcus jannaschii</i> (mj), <i>Pyrococcus abyssi</i>
 (pa), <i>Pyrobaculum aerophilum</i> (pyae), <i>Pyrococcus
 horikoshii</i> (ph), <i>Pyrococcus Furiosus</i> (pfu),
 <i>Sulfolobus solfataricus</i> P2 (ssp2), <i>Sulfolobus
 tokodaii</i> (sto), <i>Thermoplasma acidophilum</i> (ta),
 <i>Thermoplasma volcanium</i> (tv). Bacteria (35 genomes)&nbsp;:
 <i>Agrobacterium tumefaciens</i> (agrt)<i>, Aquifex aeolicus</i>
 (ae), <i>Bacillus subtilis</i> (bs), <i>Bascillus halodurans</i>
 (bh), <i>Borrelia burgdorferi</i> (bb), <i>Buchnera sp.</i> (b),
 <i>Campylobacter jejuni</i> (cj), <i>Chlamydia pneumoniae</i> (cp),
 <i>Chlamydia trachomatis</i> (ct), <i>Deinococcus radiodurans</i>
 (dr), <i>Escherichia coli</i> (ec), <i>Haemophilus influenzae</i>
 (hi), <i>Helicobacter pylori</i> (hp<i>), Listeria innocua</i>
 (lin), <i>Listeria monocytogenes</i> (lmo)<i>, Mesorhizobium
 loti</i> (mm), <i>Mycobacterium tuberculosis</i> H37R (mt), <i>M.
 tuberculosis</i> CDC 1551 (mtc), <i>Mycoplasma genitalium</i> (mg),
 <i>Mycobacterium leprae</i> (ml), <i>Mycoplasma pneumoniae</i>
 (mp), <i>Neisseria meningitidis</i> (nm), <i>Pseudomonas
 aeruginosa</i> (pae), <i>Rickettsia prowazekii</i> (rp),
 <i>Salmonella Typhi</i> (sty), <i>Sinorhizobium meliloti</i>
 (sm<i>), Staphylococcus aureus Mu50</i> (samu50), <i>Staphylococcus
 aureus</i> N315 (san315), <i>Streptococcus pyogenes M1</i> (spy),
 <i>Synechosystis sp.</i> (ssp), <i>Thermotoga maritima</i> (tm),
 <i>Treponema pallidum</i> (tp), <i>Vibrio cholerae</i> (vc),
 <i>Xylella fastidiosa</i> (xf), <i>Yersinia pestis</i> (yp).</p>
 <p>Charged amino-acids (&lsquo;char&rsquo;), polar/uncharged
 amino-acids (&lsquo;pol&rsquo;) and hydrophobic amino-acids
 (&lsquo;hyd&rsquo;) are considered in the analysis as illustrative
 variables. On the same plane, the hyperthermophilic (HTH),
 thermophilic (TH) and mesophilic (MES) pro&shy;files are reported
 as illustrative observations, along with the profiles associated
 with genes conserved (see Materials and Methods) in the various
 Eukaryal (E), Archaeal (A) and Bacterial (B) species (and the
 corresponding intersections EA, EB, AB and EAB).</p></div>
 <div><br clear="all">
  
 <hr align="left" size="1" width="33%">
 <div id="ftn1">
 <p><a href="#_ftnref1" name="_ftn1" title="">[1]</a> Unit&eacute;
 de G&eacute;n&eacute;tique Mol&eacute;culaire des Levures (URA 2171
 CNRS and UFR927, Univ. P. M. Curie, Paris), Institut Pasteur, 25,
 Rue du Dr Roux, F-75724 Paris Cedex 15, France</p></div>
 <div id="ftn2">
 <p><a href="#_ftnref2" name="_ftn2" title="">[2]</a> Centre de
 Bioinformatique, G&eacute;nopole, Institut Pasteur, 25, Rue du Dr
 Roux, F-75724 Paris Cedex 15</p></div></div>
 </body>
 </html>
